# Mask R-CNN - Analyzing the data

## Importing libraries

In [ ]:
from google.colab import drive
drive.mount('/content/drive')

Mounted at /content/drive


In [ ]:
import os
import numpy as np

from skimage.transform import resize
from tqdm.notebook import tqdm #progress bar
from tensorflow.keras.preprocessing.image import load_img

## Configurations

In [ ]:
IM_HEIGHT = 256
IM_WIDTH = 256
DATASET_PATH = "/content/drive/MyDrive/skin_cancer/assets/data_aug_dataset_complete/"

In [ ]:
task2_train_input = f"{DATASET_PATH}ISIC2018_Task1-2_Training_Input" #DATA_IMAGES
task2_val_input = f"{DATASET_PATH}ISIC2018_Task1-2_Validation_Input" #VAL_IMAGES
# task2_test_input = f"{DATASET_PATH}ISIC2018_Task1_and_2_Test_Input" #TEST_IMAGES

task2_train_ids = next(os.walk(task2_train_input))[2]
task2_val_ids= next(os.walk(task2_val_input))[2]
# task2_test_ids= next(os.walk(task2_test_input))[2]

attributes = ["_attribute_globules", "_attribute_milia_like_cyst", "_attribute_negative_network", "_attribute_pigment_network", "_attribute_streaks"]

print(f"No of images in the task2_train_input: {len(task2_train_ids)}")
print(f"No of images in the task2_val_input: {len(task2_val_ids)}")
# print(f"No of images in the task2_test_input: {len(task2_test_ids)}")
print(f"No of attributes: {len(attributes)}")

No of images in the task2_train_input: 2199
No of images in the task2_val_input: 72
No of attributes: 5


## Counts attributes of masks

In [ ]:
def count_attributes(img_ids, data_masks, flag="_segmentation"):
    occurrence_list = []

    for n, ids in enumerate(tqdm(img_ids)):
        mask = load_img(os.path.join(data_masks, ids.split(".")[0] + flag + ".png")) #load the mask
        clrs = mask.getcolors()

        if len(clrs) != 1: #if there is more than one color it is because it occurs
          occurrence_list.append(ids)

    return occurrence_list

### Training mask

In [ ]:
attributes = ["_attribute_pigment_network", "_attribute_globules", "_attribute_milia_like_cyst", "_attribute_negative_network", "_attribute_streaks"]

task2_train_gt = f"{DATASET_PATH}ISIC2018_Task2_Training_GroundTruth_v3" #DATA_MASKS

for attribute in attributes:
    print(f"attribute: {attribute}")
    occurrence_list = count_attributes(img_ids=task2_train_ids, data_masks=task2_train_gt, flag=attribute)

    print(f"occurrence_list: {occurrence_list}")
    print(f"len(occurrence_list): {len(occurrence_list)}")


attribute: _attribute_pigment_network


  0%|          | 0/2199 [00:00<?, ?it/s]

occurrence_list: ['ISIC_0011224.jpg', 'ISIC_0000231.jpg', 'ISIC_0000003.jpg', 'ISIC_0011223.jpg', 'ISIC_0000341.jpg', 'ISIC_0000261.jpg', 'ISIC_0000201.jpg', 'ISIC_0011300.jpg', 'ISIC_0011208.jpg', 'ISIC_0000217.jpg', 'ISIC_0000259.jpg', 'ISIC_0000336.jpg', 'ISIC_0000250.jpg', 'ISIC_0000236.jpg', 'ISIC_0000218.jpg', 'ISIC_0000240.jpg', 'ISIC_0000216.jpg', 'ISIC_0011205.jpg', 'ISIC_0011292.jpg', 'ISIC_0011214.jpg', 'ISIC_0000239.jpg', 'ISIC_0000228.jpg', 'ISIC_0000150.jpg', 'ISIC_0000143.jpg', 'ISIC_0000142.jpg', 'ISIC_0000191.jpg', 'ISIC_0000137.jpg', 'ISIC_0000128.jpg', 'ISIC_0000152.jpg', 'ISIC_0000138.jpg', 'ISIC_0000139.jpg', 'ISIC_0000160.jpg', 'ISIC_0000233.jpg', 'ISIC_0000171.jpg', 'ISIC_0000159.jpg', 'ISIC_0011326.jpg', 'ISIC_0011173.jpg', 'ISIC_0000129.jpg', 'ISIC_0000167.jpg', 'ISIC_0000190.jpg', 'ISIC_0000151.jpg', 'ISIC_0000202.jpg', 'ISIC_0000234.jpg', 'ISIC_0000200.jpg', 'ISIC_0000199.jpg', 'ISIC_0000203.jpg', 'ISIC_0000176.jpg', 'ISIC_0000133.jpg', 'ISIC_0000170.jpg', 'I

  0%|          | 0/2199 [00:00<?, ?it/s]

occurrence_list: ['ISIC_0000231.jpg', 'ISIC_0000341.jpg', 'ISIC_0000261.jpg', 'ISIC_0011300.jpg', 'ISIC_0011208.jpg', 'ISIC_0000259.jpg', 'ISIC_0000336.jpg', 'ISIC_0000250.jpg', 'ISIC_0000236.jpg', 'ISIC_0000218.jpg', 'ISIC_0000240.jpg', 'ISIC_0011225.jpg', 'ISIC_0000216.jpg', 'ISIC_0011205.jpg', 'ISIC_0011214.jpg', 'ISIC_0011315.jpg', 'ISIC_0011213.jpg', 'ISIC_0011207.jpg', 'ISIC_0000228.jpg', 'ISIC_0000150.jpg', 'ISIC_0011210.jpg', 'ISIC_0000143.jpg', 'ISIC_0000147.jpg', 'ISIC_0000142.jpg', 'ISIC_0000191.jpg', 'ISIC_0011203.jpg', 'ISIC_0000137.jpg', 'ISIC_0000128.jpg', 'ISIC_0000139.jpg', 'ISIC_0000160.jpg', 'ISIC_0000233.jpg', 'ISIC_0000171.jpg', 'ISIC_0000159.jpg', 'ISIC_0000129.jpg', 'ISIC_0000113.jpg', 'ISIC_0000190.jpg', 'ISIC_0000151.jpg', 'ISIC_0000202.jpg', 'ISIC_0000200.jpg', 'ISIC_0000199.jpg', 'ISIC_0000140.jpg', 'ISIC_0011328.jpg', 'ISIC_0000187.jpg', 'ISIC_0000221.jpg', 'ISIC_0000175.jpg', 'ISIC_0000148.jpg', 'ISIC_0000146.jpg', 'ISIC_0000165.jpg', 'ISIC_0000189.jpg', 'I

  0%|          | 0/2199 [00:00<?, ?it/s]

occurrence_list: ['ISIC_0011224.jpg', 'ISIC_0011322.jpg', 'ISIC_0000003.jpg', 'ISIC_0011300.jpg', 'ISIC_0000217.jpg', 'ISIC_0011292.jpg', 'ISIC_0011315.jpg', 'ISIC_0000239.jpg', 'ISIC_0000228.jpg', 'ISIC_0000125.jpg', 'ISIC_0000152.jpg', 'ISIC_0000233.jpg', 'ISIC_0011173.jpg', 'ISIC_0000234.jpg', 'ISIC_0000176.jpg', 'ISIC_0000014.jpg', 'ISIC_0011171.jpg', 'ISIC_0011156.jpg', 'ISIC_0000117.jpg', 'ISIC_0011202.jpg', 'ISIC_0011090.jpg', 'ISIC_0011104.jpg', 'ISIC_0010861.jpg', 'ISIC_0011161.jpg', 'ISIC_0011088.jpg', 'ISIC_0011137.jpg', 'ISIC_0010596.jpg', 'ISIC_0010604.jpg', 'ISIC_0010459.jpg', 'ISIC_0010475.jpg', 'ISIC_0010576.jpg', 'ISIC_0010463.jpg', 'ISIC_0010477.jpg', 'ISIC_0010857.jpg', 'ISIC_0010860.jpg', 'ISIC_0010602.jpg', 'ISIC_0010603.jpg', 'ISIC_0010570.jpg', 'ISIC_0010598.jpg', 'ISIC_0011123.jpg', 'ISIC_0010558.jpg', 'ISIC_0010562.jpg', 'ISIC_0010494.jpg', 'ISIC_0010365.jpg', 'ISIC_0010374.jpg', 'ISIC_0011099.jpg', 'ISIC_0010552.jpg', 'ISIC_0010572.jpg', 'ISIC_0011079.jpg', 'I

  0%|          | 0/2199 [00:00<?, ?it/s]

occurrence_list: ['ISIC_0011322.jpg', 'ISIC_0014144_aug_4.jpg', 'ISIC_0012768_aug_3.jpg', 'ISIC_0011223.jpg', 'ISIC_0012768_aug_9.jpg', 'ISIC_0012768_aug_8.jpg', 'ISIC_0015625_aug_5.jpg', 'ISIC_0015625_aug_0.jpg', 'ISIC_0011329.jpg', 'ISIC_0015625_aug_11.jpg', 'ISIC_0015625_aug_3.jpg', 'ISIC_0015625_aug_10.jpg', 'ISIC_0011317.jpg', 'ISIC_0015625_aug_9.jpg', 'ISIC_0015625_aug_6.jpg', 'ISIC_0010364_aug_7.jpg', 'ISIC_0012318_aug_8.jpg', 'ISIC_0013425_aug_11.jpg', 'ISIC_0015625_aug_4.jpg', 'ISIC_0012318_aug_2.jpg', 'ISIC_0000152.jpg', 'ISIC_0010364_aug_0.jpg', 'ISIC_0000160.jpg', 'ISIC_0010364_aug_8.jpg', 'ISIC_0010364_aug_11.jpg', 'ISIC_0015625_aug_2.jpg', 'ISIC_0011326.jpg', 'ISIC_0013425_aug_9.jpg', 'ISIC_0012318_aug_11.jpg', 'ISIC_0010364_aug_6.jpg', 'ISIC_0015625_aug_8.jpg', 'ISIC_0012318_aug_9.jpg', 'ISIC_0012318_aug_5.jpg', 'ISIC_0012318_aug_7.jpg', 'ISIC_0012318_aug_10.jpg', 'ISIC_0013425_aug_6.jpg', 'ISIC_0013425_aug_1.jpg', 'ISIC_0010364_aug_3.jpg', 'ISIC_0013425_aug_8.jpg', 'ISI

  0%|          | 0/2199 [00:00<?, ?it/s]

occurrence_list: ['ISIC_0000201.jpg', 'ISIC_0000240.jpg', 'ISIC_0011213.jpg', 'ISIC_0000239.jpg', 'ISIC_0011210.jpg', 'ISIC_0000198.jpg', 'ISIC_0000138.jpg', 'ISIC_0000159.jpg', 'ISIC_0000167.jpg', 'ISIC_0000200.jpg', 'ISIC_0000199.jpg', 'ISIC_0000210.jpg', 'ISIC_0000170.jpg', 'ISIC_0000140.jpg', 'ISIC_0000161.jpg', 'ISIC_0000175.jpg', 'ISIC_0000148.jpg', 'ISIC_0000165.jpg', 'ISIC_0000098.jpg', 'ISIC_0000119.jpg', 'ISIC_0000034.jpg', 'ISIC_0000155.jpg', 'ISIC_0011143.jpg', 'ISIC_0000127.jpg', 'ISIC_0000103.jpg', 'ISIC_0011095.jpg', 'ISIC_0010444.jpg', 'ISIC_0010587.jpg', 'ISIC_0010599.jpg', 'ISIC_0010493.jpg', 'ISIC_0010234.jpg', 'ISIC_0010255.jpg', 'ISIC_0010176.jpg', 'ISIC_0010244.jpg', 'ISIC_0010256.jpg', 'ISIC_0010175.jpg', 'ISIC_0010220.jpg', 'ISIC_0010070.jpg', 'ISIC_0010233.jpg', 'ISIC_0010329.jpg', 'ISIC_0009998.jpg', 'ISIC_0010251.jpg', 'ISIC_0010024.jpg', 'ISIC_0010016.jpg', 'ISIC_0009936.jpg', 'ISIC_0009937.jpg', 'ISIC_0014853.jpg', 'ISIC_0014823.jpg', 'ISIC_0014790.jpg', 'I

### Validation mask

In [ ]:
attributes = ["_attribute_pigment_network", "_attribute_globules", "_attribute_milia_like_cyst", "_attribute_negative_network", "_attribute_streaks"]

task2_val_gt = f"{DATASET_PATH}ISIC2018_Task2_Validation_GroundTruth" #VAL_MASKS

for attribute in attributes:
    print(f"attribute: {attribute}")
    occurrence_list = count_attributes(img_ids=task2_val_ids, data_masks=task2_val_gt, flag=attribute)

    print(f"occurrence_list: {occurrence_list}")
    print(f"len(occurrence_list): {len(occurrence_list)}")

attribute: _attribute_pigment_network


  0%|          | 0/72 [00:00<?, ?it/s]

occurrence_list: ['ISIC_0012585.jpg', 'ISIC_0023904.jpg', 'ISIC_0036321.jpg', 'ISIC_0016714.jpg', 'ISIC_0024135.jpg', 'ISIC_0017474.jpg', 'ISIC_0020999.jpg', 'ISIC_0016351.jpg', 'ISIC_0019334.jpg', 'ISIC_0015351.jpg', 'ISIC_0036085.jpg', 'ISIC_0023628.jpg', 'ISIC_0036147.jpg', 'ISIC_0015370.jpg', 'ISIC_0012576.jpg', 'ISIC_0023508.jpg', 'ISIC_0012633.jpg', 'ISIC_0036291.jpg', 'ISIC_0015634.jpg', 'ISIC_0021152.jpg', 'ISIC_0036174.jpg']
len(occurrence_list): 21
attribute: _attribute_globules


  0%|          | 0/72 [00:00<?, ?it/s]

occurrence_list: ['ISIC_0020233_aug_0.jpg', 'ISIC_0036236_aug_0.jpg', 'ISIC_0012585.jpg', 'ISIC_0023904.jpg', 'ISIC_0017460.jpg', 'ISIC_0036321.jpg', 'ISIC_0016714.jpg', 'ISIC_0024135.jpg', 'ISIC_0020999.jpg', 'ISIC_0016351.jpg', 'ISIC_0036236.jpg', 'ISIC_0019334.jpg', 'ISIC_0015351.jpg', 'ISIC_0020233.jpg', 'ISIC_0020953.jpg', 'ISIC_0022039.jpg', 'ISIC_0023508.jpg', 'ISIC_0012633.jpg', 'ISIC_0015634.jpg', 'ISIC_0021152.jpg', 'ISIC_0036174.jpg']
len(occurrence_list): 21
attribute: _attribute_milia_like_cyst


  0%|          | 0/72 [00:00<?, ?it/s]

occurrence_list: ['ISIC_0015552_aug_0.jpg', 'ISIC_0015552_aug_9.jpg', 'ISIC_0015552_aug_11.jpg', 'ISIC_0015552_aug_6.jpg', 'ISIC_0015552_aug_7.jpg', 'ISIC_0015552_aug_3.jpg', 'ISIC_0015552_aug_13.jpg', 'ISIC_0015552_aug_2.jpg', 'ISIC_0015552_aug_5.jpg', 'ISIC_0015552_aug_4.jpg', 'ISIC_0015552_aug_12.jpg', 'ISIC_0015552_aug_8.jpg', 'ISIC_0015552_aug_10.jpg', 'ISIC_0015552_aug_1.jpg', 'ISIC_0017474.jpg', 'ISIC_0036237.jpg', 'ISIC_0020953.jpg', 'ISIC_0022039.jpg', 'ISIC_0015462.jpg', 'ISIC_0015552.jpg']
len(occurrence_list): 20
attribute: _attribute_negative_network


  0%|          | 0/72 [00:00<?, ?it/s]

occurrence_list: ['ISIC_0021202_aug_9.jpg', 'ISIC_0021202_aug_1.jpg', 'ISIC_0021202_aug_5.jpg', 'ISIC_0021202_aug_7.jpg', 'ISIC_0021202_aug_3.jpg', 'ISIC_0021202_aug_8.jpg', 'ISIC_0021202_aug_10.jpg', 'ISIC_0021202_aug_4.jpg', 'ISIC_0021202_aug_2.jpg', 'ISIC_0021202_aug_6.jpg', 'ISIC_0021202_aug_0.jpg', 'ISIC_0021202.jpg', 'ISIC_0036237.jpg', 'ISIC_0036085.jpg', 'ISIC_0023628.jpg', 'ISIC_0036147.jpg', 'ISIC_0020953.jpg', 'ISIC_0015370.jpg', 'ISIC_0036291.jpg', 'ISIC_0015462.jpg']
len(occurrence_list): 20
attribute: _attribute_streaks


  0%|          | 0/72 [00:00<?, ?it/s]

occurrence_list: ['ISIC_0019794_aug_4.jpg', 'ISIC_0019794_aug_3.jpg', 'ISIC_0019794_aug_1.jpg', 'ISIC_0019794_aug_10.jpg', 'ISIC_0019794_aug_11.jpg', 'ISIC_0019794_aug_6.jpg', 'ISIC_0019794_aug_7.jpg', 'ISIC_0019794_aug_13.jpg', 'ISIC_0019794_aug_12.jpg', 'ISIC_0019794_aug_0.jpg', 'ISIC_0019794_aug_8.jpg', 'ISIC_0019794_aug_2.jpg', 'ISIC_0019794_aug_9.jpg', 'ISIC_0019794_aug_5.jpg', 'ISIC_0017460.jpg', 'ISIC_0036321.jpg', 'ISIC_0019794.jpg', 'ISIC_0022039.jpg', 'ISIC_0012576.jpg', 'ISIC_0023508.jpg']
len(occurrence_list): 20


### Test mask

In [ ]:
# attributes = ["_attribute_pigment_network", "_attribute_globules", "_attribute_milia_like_cyst", "_attribute_negative_network", "_attribute_streaks"]

# task2_test_gt = f"{DATASET_PATH}ISIC2018_Task2_Test_GroundTruth" #VAL_MASKS

# for attribute in attributes:
#     print(f"attribute: {attribute}")
#     occurrence_list = count_attributes(img_ids=task2_test_ids, data_masks=task2_test_gt, flag=attribute)

#     print(f"occurrence_list: {occurrence_list}")
#     print(f"len(occurrence_list): {len(occurrence_list)}")

## Mapped images

### Training

In [ ]:
training_attribute_globules = ['ISIC_0013275.jpg', 'ISIC_0013302.jpg', 'ISIC_0013311.jpg', 'ISIC_0013320.jpg', 'ISIC_0013342.jpg', 'ISIC_0013326.jpg', 'ISIC_0013390.jpg', 'ISIC_0013383.jpg', 'ISIC_0013395.jpg', 'ISIC_0013414.jpg', 'ISIC_0013399.jpg', 'ISIC_0013429.jpg', 'ISIC_0013427.jpg', 'ISIC_0013433.jpg', 'ISIC_0013417.jpg', 'ISIC_0013456.jpg', 'ISIC_0013458.jpg', 'ISIC_0013488.jpg', 'ISIC_0013501.jpg', 'ISIC_0013495.jpg', 'ISIC_0013499.jpg', 'ISIC_0013526.jpg', 'ISIC_0013525.jpg', 'ISIC_0013592.jpg', 'ISIC_0013602.jpg', 'ISIC_0013603.jpg', 'ISIC_0013600.jpg', 'ISIC_0013610.jpg', 'ISIC_0013634.jpg', 'ISIC_0013636.jpg', 'ISIC_0013652.jpg', 'ISIC_0013639.jpg', 'ISIC_0013676.jpg', 'ISIC_0013688.jpg', 'ISIC_0013737.jpg', 'ISIC_0013740.jpg', 'ISIC_0013738.jpg', 'ISIC_0013712.jpg', 'ISIC_0013758.jpg', 'ISIC_0013766.jpg', 'ISIC_0013792.jpg', 'ISIC_0013831.jpg', 'ISIC_0013837.jpg', 'ISIC_0013839.jpg', 'ISIC_0013922.jpg', 'ISIC_0013935.jpg', 'ISIC_0013948.jpg', 'ISIC_0013983.jpg', 'ISIC_0013996.jpg', 'ISIC_0013988.jpg', 'ISIC_0014029.jpg', 'ISIC_0014026.jpg', 'ISIC_0014046.jpg', 'ISIC_0014108.jpg', 'ISIC_0014099.jpg', 'ISIC_0014127.jpg', 'ISIC_0014160.jpg', 'ISIC_0014174.jpg', 'ISIC_0014173.jpg', 'ISIC_0014195.jpg', 'ISIC_0014238.jpg', 'ISIC_0014270.jpg', 'ISIC_0014274.jpg', 'ISIC_0014290.jpg', 'ISIC_0014360.jpg', 'ISIC_0014428.jpg', 'ISIC_0014431.jpg', 'ISIC_0014440.jpg', 'ISIC_0014490.jpg', 'ISIC_0014486.jpg', 'ISIC_0014476.jpg', 'ISIC_0014498.jpg', 'ISIC_0014527.jpg', 'ISIC_0014525.jpg', 'ISIC_0014661.jpg', 'ISIC_0014663.jpg', 'ISIC_0014675.jpg', 'ISIC_0014666.jpg', 'ISIC_0014680.jpg', 'ISIC_0014687.jpg', 'ISIC_0014683.jpg', 'ISIC_0014684.jpg', 'ISIC_0014702.jpg', 'ISIC_0014694.jpg', 'ISIC_0014714.jpg', 'ISIC_0014708.jpg', 'ISIC_0014729.jpg', 'ISIC_0014728.jpg', 'ISIC_0014730.jpg', 'ISIC_0014745.jpg', 'ISIC_0014746.jpg', 'ISIC_0014754.jpg', 'ISIC_0014755.jpg', 'ISIC_0014762.jpg', 'ISIC_0014773.jpg', 'ISIC_0014779.jpg', 'ISIC_0014786.jpg', 'ISIC_0014784.jpg', 'ISIC_0014794.jpg', 'ISIC_0014795.jpg', 'ISIC_0014788.jpg', 'ISIC_0014798.jpg', 'ISIC_0014805.jpg', 'ISIC_0014826.jpg', 'ISIC_0014815.jpg', 'ISIC_0014843.jpg', 'ISIC_0014836.jpg', 'ISIC_0014835.jpg', 'ISIC_0014853.jpg', 'ISIC_0014851.jpg', 'ISIC_0014867.jpg', 'ISIC_0014872.jpg', 'ISIC_0014897.jpg', 'ISIC_0014883.jpg', 'ISIC_0014891.jpg', 'ISIC_0014907.jpg', 'ISIC_0014904.jpg', 'ISIC_0014901.jpg', 'ISIC_0014910.jpg', 'ISIC_0014926.jpg', 'ISIC_0014922.jpg', 'ISIC_0014925.jpg', 'ISIC_0014928.jpg', 'ISIC_0014937.jpg', 'ISIC_0014942.jpg', 'ISIC_0014947.jpg', 'ISIC_0014964.jpg', 'ISIC_0014961.jpg', 'ISIC_0014994.jpg', 'ISIC_0014989.jpg', 'ISIC_0014987.jpg', 'ISIC_0014982.jpg', 'ISIC_0015005.jpg', 'ISIC_0015007.jpg', 'ISIC_0015008.jpg', 'ISIC_0015011.jpg', 'ISIC_0015016.jpg', 'ISIC_0015018.jpg', 'ISIC_0015009.jpg', 'ISIC_0015037.jpg', 'ISIC_0015043.jpg', 'ISIC_0015057.jpg', 'ISIC_0015056.jpg', 'ISIC_0015078.jpg', 'ISIC_0015102.jpg', 'ISIC_0015124.jpg', 'ISIC_0015129.jpg', 'ISIC_0015130.jpg', 'ISIC_0015139.jpg', 'ISIC_0015142.jpg', 'ISIC_0015146.jpg', 'ISIC_0015157.jpg', 'ISIC_0015161.jpg', 'ISIC_0015160.jpg', 'ISIC_0015180.jpg', 'ISIC_0015176.jpg', 'ISIC_0015175.jpg', 'ISIC_0015193.jpg', 'ISIC_0015204.jpg', 'ISIC_0015201.jpg', 'ISIC_0015216.jpg', 'ISIC_0015206.jpg', 'ISIC_0015223.jpg', 'ISIC_0015229.jpg', 'ISIC_0015241.jpg', 'ISIC_0015237.jpg', 'ISIC_0015250.jpg', 'ISIC_0015245.jpg', 'ISIC_0015255.jpg', 'ISIC_0015274.jpg', 'ISIC_0015264.jpg', 'ISIC_0015276.jpg', 'ISIC_0015293.jpg', 'ISIC_0015279.jpg', 'ISIC_0015310.jpg', 'ISIC_0015312.jpg', 'ISIC_0015355.jpg', 'ISIC_0015331.jpg', 'ISIC_0015357.jpg', 'ISIC_0015360.jpg', 'ISIC_0015369.jpg', 'ISIC_0015401.jpg', 'ISIC_0015411.jpg', 'ISIC_0015412.jpg', 'ISIC_0015419.jpg', 'ISIC_0015455.jpg', 'ISIC_0015485.jpg', 'ISIC_0015483.jpg', 'ISIC_0015631.jpg', 'ISIC_0015937.jpg', 'ISIC_0015950.jpg', 'ISIC_0015965.jpg', 'ISIC_0015976.jpg', 'ISIC_0015985.jpg', 'ISIC_0015980.jpg', 'ISIC_0015989.jpg', 'ISIC_0015997.jpg', 'ISIC_0015994.jpg', 'ISIC_0015993.jpg', 'ISIC_0016007.jpg', 'ISIC_0016003.jpg', 'ISIC_0016017.jpg', 'ISIC_0016014.jpg', 'ISIC_0016051.jpg', 'ISIC_0016044.jpg', 'ISIC_0016055.jpg', 'ISIC_0016060.jpg', 'ISIC_0016069.jpg', 'ISIC_0008029.jpg', 'ISIC_0007557.jpg', 'ISIC_0008294.jpg', 'ISIC_0008396.jpg', 'ISIC_0008541.jpg', 'ISIC_0008807.jpg', 'ISIC_0008913.jpg', 'ISIC_0008998.jpg', 'ISIC_0009078.jpg', 'ISIC_0009504.jpg', 'ISIC_0009869.jpg', 'ISIC_0009800.jpg', 'ISIC_0009891.jpg', 'ISIC_0009895.jpg', 'ISIC_0009889.jpg', 'ISIC_0009883.jpg', 'ISIC_0009875.jpg', 'ISIC_0009885.jpg', 'ISIC_0009884.jpg', 'ISIC_0009918.jpg', 'ISIC_0009914.jpg', 'ISIC_0009933.jpg', 'ISIC_0009929.jpg', 'ISIC_0009927.jpg', 'ISIC_0009936.jpg', 'ISIC_0009937.jpg', 'ISIC_0009940.jpg', 'ISIC_0009947.jpg', 'ISIC_0009950.jpg', 'ISIC_0009956.jpg', 'ISIC_0009970.jpg', 'ISIC_0009967.jpg', 'ISIC_0009974.jpg', 'ISIC_0009966.jpg', 'ISIC_0009998.jpg', 'ISIC_0009980.jpg', 'ISIC_0009992.jpg', 'ISIC_0010010.jpg', 'ISIC_0010009.jpg', 'ISIC_0010013.jpg', 'ISIC_0010023.jpg', 'ISIC_0010032.jpg', 'ISIC_0010018.jpg', 'ISIC_0010034.jpg', 'ISIC_0010056.jpg', 'ISIC_0010044.jpg', 'ISIC_0010041.jpg', 'ISIC_0010066.jpg', 'ISIC_0010069.jpg', 'ISIC_0010063.jpg', 'ISIC_0010061.jpg', 'ISIC_0010090.jpg', 'ISIC_0010089.jpg', 'ISIC_0010092.jpg', 'ISIC_0010088.jpg', 'ISIC_0010174.jpg', 'ISIC_0010171.jpg', 'ISIC_0010173.jpg', 'ISIC_0010175.jpg', 'ISIC_0010186.jpg', 'ISIC_0010185.jpg', 'ISIC_0010189.jpg', 'ISIC_0010206.jpg', 'ISIC_0010202.jpg', 'ISIC_0010235.jpg', 'ISIC_0010234.jpg', 'ISIC_0010233.jpg', 'ISIC_0010317.jpg', 'ISIC_0010262.jpg', 'ISIC_0010341.jpg', 'ISIC_0010349.jpg', 'ISIC_0010348.jpg', 'ISIC_0010350.jpg', 'ISIC_0010360.jpg', 'ISIC_0010441.jpg', 'ISIC_0010374.jpg', 'ISIC_0010372.jpg', 'ISIC_0010452.jpg', 'ISIC_0010463.jpg', 'ISIC_0010461.jpg', 'ISIC_0010472.jpg', 'ISIC_0010494.jpg', 'ISIC_0010493.jpg', 'ISIC_0010479.jpg', 'ISIC_0010480.jpg', 'ISIC_0010552.jpg', 'ISIC_0010557.jpg', 'ISIC_0010570.jpg', 'ISIC_0010584.jpg', 'ISIC_0010581.jpg', 'ISIC_0010571.jpg', 'ISIC_0010597.jpg', 'ISIC_0010604.jpg', 'ISIC_0010602.jpg', 'ISIC_0010598.jpg', 'ISIC_0010849.jpg', 'ISIC_0010606.jpg', 'ISIC_0010846.jpg', 'ISIC_0010850.jpg', 'ISIC_0010844.jpg', 'ISIC_0010847.jpg', 'ISIC_0011079.jpg', 'ISIC_0011083.jpg', 'ISIC_0011099.jpg', 'ISIC_0011097.jpg', 'ISIC_0011105.jpg', 'ISIC_0011101.jpg', 'ISIC_0011102.jpg', 'ISIC_0011110.jpg', 'ISIC_0011112.jpg', 'ISIC_0011115.jpg', 'ISIC_0011107.jpg', 'ISIC_0011119.jpg', 'ISIC_0011146.jpg', 'ISIC_0011159.jpg', 'ISIC_0011157.jpg', 'ISIC_0011155.jpg', 'ISIC_0011163.jpg', 'ISIC_0011168.jpg', 'ISIC_0011203.jpg', 'ISIC_0011176.jpg', 'ISIC_0011205.jpg', 'ISIC_0011220.jpg', 'ISIC_0011214.jpg', 'ISIC_0011213.jpg', 'ISIC_0011208.jpg', 'ISIC_0011207.jpg', 'ISIC_0011204.jpg', 'ISIC_0011210.jpg', 'ISIC_0011225.jpg', 'ISIC_0011298.jpg', 'ISIC_0011303.jpg', 'ISIC_0011315.jpg', 'ISIC_0011300.jpg', 'ISIC_0011324.jpg', 'ISIC_0011301.jpg', 'ISIC_0011304.jpg', 'ISIC_0011338.jpg', 'ISIC_0011330.jpg', 'ISIC_0011344.jpg', 'ISIC_0011328.jpg', 'ISIC_0011343.jpg', 'ISIC_0011349.jpg', 'ISIC_0011356.jpg', 'ISIC_0011358.jpg', 'ISIC_0011373.jpg', 'ISIC_0011384.jpg', 'ISIC_0011397.jpg', 'ISIC_0012099.jpg', 'ISIC_0012135.jpg', 'ISIC_0012160.jpg', 'ISIC_0012147.jpg', 'ISIC_0012187.jpg', 'ISIC_0012203.jpg', 'ISIC_0012191.jpg', 'ISIC_0012230.jpg', 'ISIC_0012233.jpg', 'ISIC_0012303.jpg', 'ISIC_0012316.jpg', 'ISIC_0012313.jpg', 'ISIC_0012324.jpg', 'ISIC_0012376.jpg', 'ISIC_0012406.jpg', 'ISIC_0012413.jpg', 'ISIC_0012450.jpg', 'ISIC_0012442.jpg', 'ISIC_0012464.jpg', 'ISIC_0012481.jpg', 'ISIC_0012511.jpg', 'ISIC_0012496.jpg', 'ISIC_0012503.jpg', 'ISIC_0012512.jpg', 'ISIC_0012494.jpg', 'ISIC_0012655.jpg', 'ISIC_0012657.jpg', 'ISIC_0012654.jpg', 'ISIC_0012663.jpg', 'ISIC_0012669.jpg', 'ISIC_0012673.jpg', 'ISIC_0012675.jpg', 'ISIC_0012672.jpg', 'ISIC_0012710.jpg', 'ISIC_0012749.jpg', 'ISIC_0012735.jpg', 'ISIC_0012725.jpg', 'ISIC_0012758.jpg', 'ISIC_0012804.jpg', 'ISIC_0012888.jpg', 'ISIC_0012891.jpg', 'ISIC_0012883.jpg', 'ISIC_0012904.jpg', 'ISIC_0012981.jpg', 'ISIC_0012969.jpg', 'ISIC_0012950.jpg', 'ISIC_0013010.jpg', 'ISIC_0013025.jpg', 'ISIC_0013015.jpg', 'ISIC_0013045.jpg', 'ISIC_0013082.jpg', 'ISIC_0013072.jpg', 'ISIC_0013073.jpg', 'ISIC_0013090.jpg', 'ISIC_0013140.jpg', 'ISIC_0013147.jpg', 'ISIC_0013159.jpg', 'ISIC_0013172.jpg', 'ISIC_0013176.jpg', 'ISIC_0013190.jpg', 'ISIC_0013220.jpg', 'ISIC_0013219.jpg', 'ISIC_0013224.jpg', 'ISIC_0013222.jpg', 'ISIC_0013244.jpg', 'ISIC_0013242.jpg', 'ISIC_0013248.jpg', 'ISIC_0000008.jpg', 'ISIC_0000001.jpg', 'ISIC_0000017.jpg', 'ISIC_0000012.jpg', 'ISIC_0000009.jpg', 'ISIC_0000021.jpg', 'ISIC_0000013.jpg', 'ISIC_0000030.jpg', 'ISIC_0000032.jpg', 'ISIC_0000035.jpg', 'ISIC_0000034.jpg', 'ISIC_0000027.jpg', 'ISIC_0000031.jpg', 'ISIC_0000044.jpg', 'ISIC_0000047.jpg', 'ISIC_0000037.jpg', 'ISIC_0000043.jpg', 'ISIC_0000049.jpg', 'ISIC_0000042.jpg', 'ISIC_0000052.jpg', 'ISIC_0000058.jpg', 'ISIC_0000062.jpg', 'ISIC_0000053.jpg', 'ISIC_0000061.jpg', 'ISIC_0000078.jpg', 'ISIC_0000067.jpg', 'ISIC_0000064.jpg', 'ISIC_0000073.jpg', 'ISIC_0000069.jpg', 'ISIC_0000065.jpg', 'ISIC_0000081.jpg', 'ISIC_0000086.jpg', 'ISIC_0000103.jpg', 'ISIC_0000096.jpg', 'ISIC_0000105.jpg', 'ISIC_0000108.jpg', 'ISIC_0000097.jpg', 'ISIC_0000115.jpg', 'ISIC_0000113.jpg', 'ISIC_0000131.jpg', 'ISIC_0000127.jpg', 'ISIC_0000129.jpg', 'ISIC_0000128.jpg', 'ISIC_0000139.jpg', 'ISIC_0000140.jpg', 'ISIC_0000137.jpg', 'ISIC_0000143.jpg', 'ISIC_0000142.jpg', 'ISIC_0000146.jpg', 'ISIC_0000147.jpg', 'ISIC_0000150.jpg', 'ISIC_0000148.jpg', 'ISIC_0000160.jpg', 'ISIC_0000159.jpg', 'ISIC_0000163.jpg', 'ISIC_0000151.jpg', 'ISIC_0000155.jpg', 'ISIC_0000175.jpg', 'ISIC_0000165.jpg', 'ISIC_0000171.jpg', 'ISIC_0000184.jpg', 'ISIC_0000187.jpg', 'ISIC_0000190.jpg', 'ISIC_0000191.jpg', 'ISIC_0000189.jpg', 'ISIC_0000188.jpg', 'ISIC_0000199.jpg', 'ISIC_0000209.jpg', 'ISIC_0000200.jpg', 'ISIC_0000202.jpg', 'ISIC_0000215.jpg', 'ISIC_0000218.jpg', 'ISIC_0000221.jpg', 'ISIC_0000216.jpg', 'ISIC_0000236.jpg', 'ISIC_0000230.jpg', 'ISIC_0000231.jpg', 'ISIC_0000233.jpg', 'ISIC_0000228.jpg', 'ISIC_0000250.jpg', 'ISIC_0000240.jpg', 'ISIC_0000243.jpg', 'ISIC_0000260.jpg', 'ISIC_0000261.jpg', 'ISIC_0000269.jpg', 'ISIC_0000259.jpg', 'ISIC_0000271.jpg', 'ISIC_0000255.jpg', 'ISIC_0000262.jpg', 'ISIC_0000256.jpg', 'ISIC_0000282.jpg', 'ISIC_0000283.jpg', 'ISIC_0000292.jpg', 'ISIC_0000281.jpg', 'ISIC_0000290.jpg', 'ISIC_0000310.jpg', 'ISIC_0000301.jpg', 'ISIC_0000297.jpg', 'ISIC_0000319.jpg', 'ISIC_0000325.jpg', 'ISIC_0000329.jpg', 'ISIC_0000328.jpg', 'ISIC_0000341.jpg', 'ISIC_0000336.jpg', 'ISIC_0000355.jpg', 'ISIC_0000346.jpg', 'ISIC_0000370.jpg', 'ISIC_0000363.jpg', 'ISIC_0000387.jpg', 'ISIC_0000378.jpg', 'ISIC_0000385.jpg', 'ISIC_0000380.jpg', 'ISIC_0000382.jpg', 'ISIC_0000379.jpg', 'ISIC_0000407.jpg', 'ISIC_0000408.jpg', 'ISIC_0000426.jpg', 'ISIC_0000425.jpg', 'ISIC_0000415.jpg', 'ISIC_0000423.jpg', 'ISIC_0000455.jpg', 'ISIC_0000442.jpg', 'ISIC_0000443.jpg', 'ISIC_0000457.jpg', 'ISIC_0000453.jpg', 'ISIC_0000465.jpg', 'ISIC_0000474.jpg', 'ISIC_0000466.jpg', 'ISIC_0000471.jpg', 'ISIC_0000486.jpg', 'ISIC_0000490.jpg', 'ISIC_0000483.jpg', 'ISIC_0000484.jpg', 'ISIC_0000485.jpg', 'ISIC_0000493.jpg', 'ISIC_0000494.jpg', 'ISIC_0000506.jpg', 'ISIC_0000495.jpg', 'ISIC_0000496.jpg', 'ISIC_0000505.jpg', 'ISIC_0000501.jpg', 'ISIC_0000510.jpg', 'ISIC_0000522.jpg', 'ISIC_0000527.jpg', 'ISIC_0000525.jpg', 'ISIC_0000537.jpg', 'ISIC_0000533.jpg', 'ISIC_0000544.jpg', 'ISIC_0000545.jpg', 'ISIC_0001105.jpg', 'ISIC_0001118.jpg', 'ISIC_0001103.jpg', 'ISIC_0000552.jpg', 'ISIC_0001152.jpg', 'ISIC_0001119.jpg', 'ISIC_0001142.jpg', 'ISIC_0001133.jpg', 'ISIC_0001126.jpg', 'ISIC_0001242.jpg', 'ISIC_0001212.jpg', 'ISIC_0001204.jpg', 'ISIC_0001871.jpg', 'ISIC_0002107.jpg', 'ISIC_0002438.jpg', 'ISIC_0002287.jpg', 'ISIC_0002439.jpg', 'ISIC_0002836.jpg', 'ISIC_0002829.jpg', 'ISIC_0003559.jpg', 'ISIC_0003174.jpg', 'ISIC_0003539.jpg', 'ISIC_0004337.jpg', 'ISIC_0003582.jpg', 'ISIC_0003805.jpg', 'ISIC_0005620.jpg', 'ISIC_0006914.jpg', 'ISIC_0006795.jpg', 'ISIC_0006671.jpg', 'ISIC_0007332.jpg', 'ISIC_0007344.jpg']

training_attribute_milia_like_cyst = ['ISIC_0013235.jpg', 'ISIC_0013271.jpg', 'ISIC_0013277.jpg', 'ISIC_0013288.jpg', 'ISIC_0013274.jpg', 'ISIC_0013319.jpg', 'ISIC_0013320.jpg', 'ISIC_0013325.jpg', 'ISIC_0013335.jpg', 'ISIC_0013330.jpg', 'ISIC_0013356.jpg', 'ISIC_0013355.jpg', 'ISIC_0013360.jpg', 'ISIC_0013364.jpg', 'ISIC_0013371.jpg', 'ISIC_0013397.jpg', 'ISIC_0013383.jpg', 'ISIC_0013395.jpg', 'ISIC_0013374.jpg', 'ISIC_0013385.jpg', 'ISIC_0013414.jpg', 'ISIC_0013410.jpg', 'ISIC_0013405.jpg', 'ISIC_0013433.jpg', 'ISIC_0013430.jpg', 'ISIC_0013428.jpg', 'ISIC_0013432.jpg', 'ISIC_0013457.jpg', 'ISIC_0013455.jpg', 'ISIC_0013437.jpg', 'ISIC_0013458.jpg', 'ISIC_0013473.jpg', 'ISIC_0013488.jpg', 'ISIC_0013487.jpg', 'ISIC_0013489.jpg', 'ISIC_0013495.jpg', 'ISIC_0013512.jpg', 'ISIC_0013499.jpg', 'ISIC_0013493.jpg', 'ISIC_0013500.jpg', 'ISIC_0013552.jpg', 'ISIC_0013559.jpg', 'ISIC_0013558.jpg', 'ISIC_0013562.jpg', 'ISIC_0013579.jpg', 'ISIC_0013615.jpg', 'ISIC_0013618.jpg', 'ISIC_0013635.jpg', 'ISIC_0013636.jpg', 'ISIC_0013626.jpg', 'ISIC_0013621.jpg', 'ISIC_0013617.jpg', 'ISIC_0013663.jpg', 'ISIC_0013637.jpg', 'ISIC_0013639.jpg', 'ISIC_0013672.jpg', 'ISIC_0013684.jpg', 'ISIC_0013688.jpg', 'ISIC_0013685.jpg', 'ISIC_0013691.jpg', 'ISIC_0013709.jpg', 'ISIC_0013758.jpg', 'ISIC_0013767.jpg', 'ISIC_0013766.jpg', 'ISIC_0013748.jpg', 'ISIC_0013747.jpg', 'ISIC_0013778.jpg', 'ISIC_0013792.jpg', 'ISIC_0013782.jpg', 'ISIC_0013797.jpg', 'ISIC_0013794.jpg', 'ISIC_0013805.jpg', 'ISIC_0013817.jpg', 'ISIC_0013809.jpg', 'ISIC_0013831.jpg', 'ISIC_0013815.jpg', 'ISIC_0013830.jpg', 'ISIC_0013819.jpg', 'ISIC_0013837.jpg', 'ISIC_0013840.jpg', 'ISIC_0013844.jpg', 'ISIC_0013843.jpg', 'ISIC_0013833.jpg', 'ISIC_0013861.jpg', 'ISIC_0013867.jpg', 'ISIC_0013862.jpg', 'ISIC_0013911.jpg', 'ISIC_0013896.jpg', 'ISIC_0013922.jpg', 'ISIC_0013917.jpg', 'ISIC_0013910.jpg', 'ISIC_0013918.jpg', 'ISIC_0013897.jpg', 'ISIC_0013929.jpg', 'ISIC_0013942.jpg', 'ISIC_0013936.jpg', 'ISIC_0013961.jpg', 'ISIC_0013946.jpg', 'ISIC_0013975.jpg', 'ISIC_0013967.jpg', 'ISIC_0013980.jpg', 'ISIC_0013972.jpg', 'ISIC_0014001.jpg', 'ISIC_0013987.jpg', 'ISIC_0013984.jpg', 'ISIC_0013998.jpg', 'ISIC_0013996.jpg', 'ISIC_0013986.jpg', 'ISIC_0013997.jpg', 'ISIC_0013988.jpg', 'ISIC_0014028.jpg', 'ISIC_0014059.jpg', 'ISIC_0014037.jpg', 'ISIC_0014029.jpg', 'ISIC_0014032.jpg', 'ISIC_0014026.jpg', 'ISIC_0014046.jpg', 'ISIC_0014073.jpg', 'ISIC_0014069.jpg', 'ISIC_0014076.jpg', 'ISIC_0014077.jpg', 'ISIC_0014062.jpg', 'ISIC_0014110.jpg', 'ISIC_0014114.jpg', 'ISIC_0014090.jpg', 'ISIC_0014103.jpg', 'ISIC_0014108.jpg', 'ISIC_0014092.jpg', 'ISIC_0014150.jpg', 'ISIC_0014129.jpg', 'ISIC_0014132.jpg', 'ISIC_0014139.jpg', 'ISIC_0014156.jpg', 'ISIC_0014151.jpg', 'ISIC_0014164.jpg', 'ISIC_0014166.jpg', 'ISIC_0014160.jpg', 'ISIC_0014163.jpg', 'ISIC_0014211.jpg', 'ISIC_0014189.jpg', 'ISIC_0014183.jpg', 'ISIC_0014190.jpg', 'ISIC_0014181.jpg', 'ISIC_0014197.jpg', 'ISIC_0014186.jpg', 'ISIC_0014217.jpg', 'ISIC_0014219.jpg', 'ISIC_0014225.jpg', 'ISIC_0014238.jpg', 'ISIC_0014288.jpg', 'ISIC_0014289.jpg', 'ISIC_0014273.jpg', 'ISIC_0014253.jpg', 'ISIC_0014286.jpg', 'ISIC_0014274.jpg', 'ISIC_0014325.jpg', 'ISIC_0014317.jpg', 'ISIC_0014327.jpg', 'ISIC_0014308.jpg', 'ISIC_0014316.jpg', 'ISIC_0014324.jpg', 'ISIC_0014337.jpg', 'ISIC_0014347.jpg', 'ISIC_0014357.jpg', 'ISIC_0014393.jpg', 'ISIC_0014372.jpg', 'ISIC_0014382.jpg', 'ISIC_0014392.jpg', 'ISIC_0014361.jpg', 'ISIC_0014386.jpg', 'ISIC_0014366.jpg', 'ISIC_0014419.jpg', 'ISIC_0014410.jpg', 'ISIC_0014397.jpg', 'ISIC_0014430.jpg', 'ISIC_0014422.jpg', 'ISIC_0014457.jpg', 'ISIC_0014454.jpg', 'ISIC_0014475.jpg', 'ISIC_0014438.jpg', 'ISIC_0014470.jpg', 'ISIC_0014469.jpg', 'ISIC_0014440.jpg', 'ISIC_0014453.jpg', 'ISIC_0014490.jpg', 'ISIC_0014500.jpg', 'ISIC_0014480.jpg', 'ISIC_0014498.jpg', 'ISIC_0014501.jpg', 'ISIC_0014527.jpg', 'ISIC_0014522.jpg', 'ISIC_0014511.jpg', 'ISIC_0014513.jpg', 'ISIC_0014526.jpg', 'ISIC_0014503.jpg', 'ISIC_0014506.jpg', 'ISIC_0014518.jpg', 'ISIC_0014545.jpg', 'ISIC_0014529.jpg', 'ISIC_0014535.jpg', 'ISIC_0014541.jpg', 'ISIC_0014543.jpg', 'ISIC_0014537.jpg', 'ISIC_0014580.jpg', 'ISIC_0014577.jpg', 'ISIC_0014558.jpg', 'ISIC_0014583.jpg', 'ISIC_0014609.jpg', 'ISIC_0014586.jpg', 'ISIC_0014599.jpg', 'ISIC_0014605.jpg', 'ISIC_0014596.jpg', 'ISIC_0014610.jpg', 'ISIC_0014617.jpg', 'ISIC_0014603.jpg', 'ISIC_0014625.jpg', 'ISIC_0014623.jpg', 'ISIC_0014628.jpg', 'ISIC_0014624.jpg', 'ISIC_0014642.jpg', 'ISIC_0014643.jpg', 'ISIC_0014640.jpg', 'ISIC_0014658.jpg', 'ISIC_0014663.jpg', 'ISIC_0014651.jpg', 'ISIC_0014694.jpg', 'ISIC_0014707.jpg', 'ISIC_0014713.jpg', 'ISIC_0014714.jpg', 'ISIC_0014729.jpg', 'ISIC_0014728.jpg', 'ISIC_0014723.jpg', 'ISIC_0014722.jpg', 'ISIC_0014726.jpg', 'ISIC_0014724.jpg', 'ISIC_0014725.jpg', 'ISIC_0014730.jpg', 'ISIC_0014742.jpg', 'ISIC_0014754.jpg', 'ISIC_0014749.jpg', 'ISIC_0014763.jpg', 'ISIC_0014760.jpg', 'ISIC_0014775.jpg', 'ISIC_0014783.jpg', 'ISIC_0014784.jpg', 'ISIC_0014794.jpg', 'ISIC_0014795.jpg', 'ISIC_0014791.jpg', 'ISIC_0014804.jpg', 'ISIC_0014797.jpg', 'ISIC_0014806.jpg', 'ISIC_0014809.jpg', 'ISIC_0014826.jpg', 'ISIC_0014823.jpg', 'ISIC_0014830.jpg', 'ISIC_0014834.jpg', 'ISIC_0014831.jpg', 'ISIC_0014835.jpg', 'ISIC_0014839.jpg', 'ISIC_0014844.jpg', 'ISIC_0014849.jpg', 'ISIC_0014845.jpg', 'ISIC_0014854.jpg', 'ISIC_0014855.jpg', 'ISIC_0014850.jpg', 'ISIC_0014897.jpg', 'ISIC_0014904.jpg', 'ISIC_0014911.jpg', 'ISIC_0014937.jpg', 'ISIC_0014942.jpg', 'ISIC_0014943.jpg', 'ISIC_0014945.jpg', 'ISIC_0014946.jpg', 'ISIC_0014951.jpg', 'ISIC_0014947.jpg', 'ISIC_0015008.jpg', 'ISIC_0015011.jpg', 'ISIC_0015013.jpg', 'ISIC_0015031.jpg', 'ISIC_0015037.jpg', 'ISIC_0015044.jpg', 'ISIC_0015051.jpg', 'ISIC_0015050.jpg', 'ISIC_0015062.jpg', 'ISIC_0015082.jpg', 'ISIC_0015113.jpg', 'ISIC_0015108.jpg', 'ISIC_0015124.jpg', 'ISIC_0015153.jpg', 'ISIC_0015158.jpg', 'ISIC_0015160.jpg', 'ISIC_0015170.jpg', 'ISIC_0015180.jpg', 'ISIC_0015175.jpg', 'ISIC_0015200.jpg', 'ISIC_0015216.jpg', 'ISIC_0015233.jpg', 'ISIC_0015250.jpg', 'ISIC_0015258.jpg', 'ISIC_0015245.jpg', 'ISIC_0015274.jpg', 'ISIC_0015260.jpg', 'ISIC_0015295.jpg', 'ISIC_0015291.jpg', 'ISIC_0015279.jpg', 'ISIC_0015310.jpg', 'ISIC_0015363.jpg', 'ISIC_0015360.jpg', 'ISIC_0015395.jpg', 'ISIC_0015443.jpg', 'ISIC_0015631.jpg', 'ISIC_0015944.jpg', 'ISIC_0015967.jpg', 'ISIC_0015963.jpg', 'ISIC_0015965.jpg', 'ISIC_0015961.jpg', 'ISIC_0015969.jpg', 'ISIC_0015989.jpg', 'ISIC_0016000.jpg', 'ISIC_0016022.jpg', 'ISIC_0016017.jpg', 'ISIC_0016014.jpg', 'ISIC_0016041.jpg', 'ISIC_0016046.jpg', 'ISIC_0016052.jpg', 'ISIC_0016062.jpg', 'ISIC_0016070.jpg', 'ISIC_0007557.jpg', 'ISIC_0007760.jpg', 'ISIC_0007693.jpg', 'ISIC_0007796.jpg', 'ISIC_0008116.jpg', 'ISIC_0008256.jpg', 'ISIC_0008528.jpg', 'ISIC_0008280.jpg', 'ISIC_0008552.jpg', 'ISIC_0008347.jpg', 'ISIC_0009035.jpg', 'ISIC_0008879.jpg', 'ISIC_0008659.jpg', 'ISIC_0008600.jpg', 'ISIC_0009160.jpg', 'ISIC_0009344.jpg', 'ISIC_0009078.jpg', 'ISIC_0009869.jpg', 'ISIC_0009873.jpg', 'ISIC_0009800.jpg', 'ISIC_0009758.jpg', 'ISIC_0009599.jpg', 'ISIC_0009883.jpg', 'ISIC_0009875.jpg', 'ISIC_0009910.jpg', 'ISIC_0009917.jpg', 'ISIC_0009915.jpg', 'ISIC_0009901.jpg', 'ISIC_0009899.jpg', 'ISIC_0009906.jpg', 'ISIC_0009904.jpg', 'ISIC_0009931.jpg', 'ISIC_0009921.jpg', 'ISIC_0009940.jpg', 'ISIC_0009942.jpg', 'ISIC_0009944.jpg', 'ISIC_0009939.jpg', 'ISIC_0009961.jpg', 'ISIC_0009949.jpg', 'ISIC_0009956.jpg', 'ISIC_0009948.jpg', 'ISIC_0009970.jpg', 'ISIC_0009967.jpg', 'ISIC_0009974.jpg', 'ISIC_0009969.jpg', 'ISIC_0009990.jpg', 'ISIC_0009994.jpg', 'ISIC_0009980.jpg', 'ISIC_0009995.jpg', 'ISIC_0010017.jpg', 'ISIC_0010005.jpg', 'ISIC_0010021.jpg', 'ISIC_0010022.jpg', 'ISIC_0010047.jpg', 'ISIC_0010056.jpg', 'ISIC_0010042.jpg', 'ISIC_0010037.jpg', 'ISIC_0010041.jpg', 'ISIC_0010070.jpg', 'ISIC_0010062.jpg', 'ISIC_0010100.jpg', 'ISIC_0010092.jpg', 'ISIC_0010074.jpg', 'ISIC_0010176.jpg', 'ISIC_0010180.jpg', 'ISIC_0010189.jpg', 'ISIC_0010187.jpg', 'ISIC_0010201.jpg', 'ISIC_0010206.jpg', 'ISIC_0010213.jpg', 'ISIC_0010204.jpg', 'ISIC_0010222.jpg', 'ISIC_0010238.jpg', 'ISIC_0010229.jpg', 'ISIC_0010232.jpg', 'ISIC_0010236.jpg', 'ISIC_0010233.jpg', 'ISIC_0010249.jpg', 'ISIC_0010248.jpg', 'ISIC_0010239.jpg', 'ISIC_0010264.jpg', 'ISIC_0010323.jpg', 'ISIC_0010333.jpg', 'ISIC_0010334.jpg', 'ISIC_0010335.jpg', 'ISIC_0010347.jpg', 'ISIC_0010349.jpg', 'ISIC_0010351.jpg', 'ISIC_0010365.jpg', 'ISIC_0010370.jpg', 'ISIC_0010374.jpg', 'ISIC_0010455.jpg', 'ISIC_0010475.jpg', 'ISIC_0010463.jpg', 'ISIC_0010461.jpg', 'ISIC_0010459.jpg', 'ISIC_0010494.jpg', 'ISIC_0010483.jpg', 'ISIC_0010496.jpg', 'ISIC_0010490.jpg', 'ISIC_0010477.jpg', 'ISIC_0010562.jpg', 'ISIC_0010558.jpg', 'ISIC_0010552.jpg', 'ISIC_0010557.jpg', 'ISIC_0010570.jpg', 'ISIC_0010572.jpg', 'ISIC_0010576.jpg', 'ISIC_0010604.jpg', 'ISIC_0010602.jpg', 'ISIC_0010603.jpg', 'ISIC_0010596.jpg', 'ISIC_0010598.jpg', 'ISIC_0010852.jpg', 'ISIC_0011079.jpg', 'ISIC_0010857.jpg', 'ISIC_0010861.jpg', 'ISIC_0010860.jpg', 'ISIC_0011099.jpg', 'ISIC_0011104.jpg', 'ISIC_0011097.jpg', 'ISIC_0011090.jpg', 'ISIC_0011105.jpg', 'ISIC_0011088.jpg', 'ISIC_0011123.jpg', 'ISIC_0011137.jpg', 'ISIC_0011161.jpg', 'ISIC_0011157.jpg', 'ISIC_0011156.jpg', 'ISIC_0011202.jpg', 'ISIC_0011170.jpg', 'ISIC_0011173.jpg', 'ISIC_0011171.jpg', 'ISIC_0011204.jpg', 'ISIC_0011297.jpg', 'ISIC_0011224.jpg', 'ISIC_0011292.jpg', 'ISIC_0011229.jpg', 'ISIC_0011228.jpg', 'ISIC_0011298.jpg', 'ISIC_0011306.jpg', 'ISIC_0011303.jpg', 'ISIC_0011315.jpg', 'ISIC_0011300.jpg', 'ISIC_0011327.jpg', 'ISIC_0011324.jpg', 'ISIC_0011322.jpg', 'ISIC_0011301.jpg', 'ISIC_0011304.jpg', 'ISIC_0011338.jpg', 'ISIC_0011330.jpg', 'ISIC_0011345.jpg', 'ISIC_0011349.jpg', 'ISIC_0011390.jpg', 'ISIC_0011382.jpg', 'ISIC_0011402.jpg', 'ISIC_0012092.jpg', 'ISIC_0012102.jpg', 'ISIC_0012108.jpg', 'ISIC_0012137.jpg', 'ISIC_0012151.jpg', 'ISIC_0012148.jpg', 'ISIC_0012187.jpg', 'ISIC_0012205.jpg', 'ISIC_0012178.jpg', 'ISIC_0012211.jpg', 'ISIC_0012230.jpg', 'ISIC_0012228.jpg', 'ISIC_0012227.jpg', 'ISIC_0012235.jpg', 'ISIC_0012261.jpg', 'ISIC_0012268.jpg', 'ISIC_0012273.jpg', 'ISIC_0012281.jpg', 'ISIC_0012309.jpg', 'ISIC_0012360.jpg', 'ISIC_0012357.jpg', 'ISIC_0012382.jpg', 'ISIC_0012415.jpg', 'ISIC_0012395.jpg', 'ISIC_0012445.jpg', 'ISIC_0012475.jpg', 'ISIC_0012526.jpg', 'ISIC_0012529.jpg', 'ISIC_0012656.jpg', 'ISIC_0012655.jpg', 'ISIC_0012548.jpg', 'ISIC_0012658.jpg', 'ISIC_0012660.jpg', 'ISIC_0012661.jpg', 'ISIC_0012662.jpg', 'ISIC_0012678.jpg', 'ISIC_0012679.jpg', 'ISIC_0012685.jpg', 'ISIC_0012684.jpg', 'ISIC_0012697.jpg', 'ISIC_0012720.jpg', 'ISIC_0012721.jpg', 'ISIC_0012715.jpg', 'ISIC_0012737.jpg', 'ISIC_0012741.jpg', 'ISIC_0012773.jpg', 'ISIC_0012786.jpg', 'ISIC_0012756.jpg', 'ISIC_0012803.jpg', 'ISIC_0012813.jpg', 'ISIC_0012828.jpg', 'ISIC_0012865.jpg', 'ISIC_0012844.jpg', 'ISIC_0012878.jpg', 'ISIC_0012840.jpg', 'ISIC_0012855.jpg', 'ISIC_0012877.jpg', 'ISIC_0012888.jpg', 'ISIC_0012880.jpg', 'ISIC_0012884.jpg', 'ISIC_0012932.jpg', 'ISIC_0012905.jpg', 'ISIC_0012907.jpg', 'ISIC_0012903.jpg', 'ISIC_0012940.jpg', 'ISIC_0012930.jpg', 'ISIC_0012904.jpg', 'ISIC_0012939.jpg', 'ISIC_0012961.jpg', 'ISIC_0012950.jpg', 'ISIC_0012957.jpg', 'ISIC_0012945.jpg', 'ISIC_0012944.jpg', 'ISIC_0012987.jpg', 'ISIC_0012986.jpg', 'ISIC_0013001.jpg', 'ISIC_0012999.jpg', 'ISIC_0013034.jpg', 'ISIC_0013032.jpg', 'ISIC_0013037.jpg', 'ISIC_0013015.jpg', 'ISIC_0013024.jpg', 'ISIC_0013035.jpg', 'ISIC_0013045.jpg', 'ISIC_0013054.jpg', 'ISIC_0013048.jpg', 'ISIC_0013082.jpg', 'ISIC_0013087.jpg', 'ISIC_0013084.jpg', 'ISIC_0013070.jpg', 'ISIC_0013072.jpg', 'ISIC_0013065.jpg', 'ISIC_0013079.jpg', 'ISIC_0013090.jpg', 'ISIC_0013114.jpg', 'ISIC_0013112.jpg', 'ISIC_0013104.jpg', 'ISIC_0013120.jpg', 'ISIC_0013109.jpg', 'ISIC_0013089.jpg', 'ISIC_0013136.jpg', 'ISIC_0013124.jpg', 'ISIC_0013141.jpg', 'ISIC_0013126.jpg', 'ISIC_0013129.jpg', 'ISIC_0013128.jpg', 'ISIC_0013140.jpg', 'ISIC_0013134.jpg', 'ISIC_0013164.jpg', 'ISIC_0013155.jpg', 'ISIC_0013165.jpg', 'ISIC_0013167.jpg', 'ISIC_0013169.jpg', 'ISIC_0013166.jpg', 'ISIC_0013163.jpg', 'ISIC_0013159.jpg', 'ISIC_0013150.jpg', 'ISIC_0013160.jpg', 'ISIC_0013184.jpg', 'ISIC_0013172.jpg', 'ISIC_0013176.jpg', 'ISIC_0013187.jpg', 'ISIC_0013193.jpg', 'ISIC_0013189.jpg', 'ISIC_0013207.jpg', 'ISIC_0013200.jpg', 'ISIC_0013229.jpg', 'ISIC_0013232.jpg', 'ISIC_0013219.jpg', 'ISIC_0013230.jpg', 'ISIC_0013222.jpg', 'ISIC_0013223.jpg', 'ISIC_0013244.jpg', 'ISIC_0013233.jpg', 'ISIC_0013242.jpg', 'ISIC_0013248.jpg', 'ISIC_0013238.jpg', 'ISIC_0000003.jpg', 'ISIC_0000014.jpg', 'ISIC_0000011.jpg', 'ISIC_0000012.jpg', 'ISIC_0000072.jpg', 'ISIC_0000067.jpg', 'ISIC_0000077.jpg', 'ISIC_0000117.jpg', 'ISIC_0000125.jpg', 'ISIC_0000152.jpg', 'ISIC_0000173.jpg', 'ISIC_0000176.jpg', 'ISIC_0000217.jpg', 'ISIC_0000234.jpg', 'ISIC_0000230.jpg', 'ISIC_0000233.jpg', 'ISIC_0000228.jpg', 'ISIC_0000239.jpg', 'ISIC_0000339.jpg', 'ISIC_0000457.jpg', 'ISIC_0000465.jpg', 'ISIC_0000474.jpg', 'ISIC_0000473.jpg', 'ISIC_0000460.jpg', 'ISIC_0000487.jpg', 'ISIC_0000481.jpg', 'ISIC_0000494.jpg', 'ISIC_0000507.jpg', 'ISIC_0000509.jpg', 'ISIC_0000525.jpg', 'ISIC_0001212.jpg', 'ISIC_0001216.jpg', 'ISIC_0001204.jpg', 'ISIC_0001254.jpg', 'ISIC_0001213.jpg', 'ISIC_0001247.jpg', 'ISIC_0001306.jpg', 'ISIC_0002251.jpg', 'ISIC_0001871.jpg', 'ISIC_0002206.jpg', 'ISIC_0002476.jpg', 'ISIC_0002469.jpg', 'ISIC_0002836.jpg', 'ISIC_0002879.jpg', 'ISIC_0002829.jpg', 'ISIC_0002806.jpg', 'ISIC_0003346.jpg', 'ISIC_0003005.jpg', 'ISIC_0003539.jpg', 'ISIC_0003582.jpg', 'ISIC_0004110.jpg', 'ISIC_0004715.jpg', 'ISIC_0004309.jpg', 'ISIC_0004115.jpg', 'ISIC_0005187.jpg', 'ISIC_0005666.jpg', 'ISIC_0006800.jpg', 'ISIC_0006350.jpg', 'ISIC_0006711.jpg', 'ISIC_0007322.jpg', 'ISIC_0007087.jpg', 'ISIC_0007038.jpg']

training_attribute_negative_network = ['ISIC_0013277.jpg', 'ISIC_0013311.jpg', 'ISIC_0013325.jpg', 'ISIC_0013395.jpg', 'ISIC_0013374.jpg', 'ISIC_0013403.jpg', 'ISIC_0013425.jpg', 'ISIC_0013491.jpg', 'ISIC_0013493.jpg', 'ISIC_0013518.jpg', 'ISIC_0013572.jpg', 'ISIC_0013594.jpg', 'ISIC_0013671.jpg', 'ISIC_0013702.jpg', 'ISIC_0013736.jpg', 'ISIC_0013793.jpg', 'ISIC_0013809.jpg', 'ISIC_0013842.jpg', 'ISIC_0013911.jpg', 'ISIC_0013948.jpg', 'ISIC_0013967.jpg', 'ISIC_0013984.jpg', 'ISIC_0014013.jpg', 'ISIC_0013995.jpg', 'ISIC_0014046.jpg', 'ISIC_0014127.jpg', 'ISIC_0014144.jpg', 'ISIC_0014133.jpg', 'ISIC_0014183.jpg', 'ISIC_0014181.jpg', 'ISIC_0014186.jpg', 'ISIC_0014238.jpg', 'ISIC_0014253.jpg', 'ISIC_0014255.jpg', 'ISIC_0014317.jpg', 'ISIC_0014347.jpg', 'ISIC_0014372.jpg', 'ISIC_0014502.jpg', 'ISIC_0014577.jpg', 'ISIC_0014663.jpg', 'ISIC_0014696.jpg', 'ISIC_0014694.jpg', 'ISIC_0014714.jpg', 'ISIC_0014755.jpg', 'ISIC_0014784.jpg', 'ISIC_0014826.jpg', 'ISIC_0014825.jpg', 'ISIC_0014830.jpg', 'ISIC_0014833.jpg', 'ISIC_0014846.jpg', 'ISIC_0014867.jpg', 'ISIC_0014862.jpg', 'ISIC_0014857.jpg', 'ISIC_0014872.jpg', 'ISIC_0014897.jpg', 'ISIC_0014891.jpg', 'ISIC_0014937.jpg', 'ISIC_0014961.jpg', 'ISIC_0014963.jpg', 'ISIC_0015043.jpg', 'ISIC_0015056.jpg', 'ISIC_0015109.jpg', 'ISIC_0015102.jpg', 'ISIC_0015142.jpg', 'ISIC_0015167.jpg', 'ISIC_0015189.jpg', 'ISIC_0015201.jpg', 'ISIC_0015224.jpg', 'ISIC_0015219.jpg', 'ISIC_0015256.jpg', 'ISIC_0015258.jpg', 'ISIC_0015255.jpg', 'ISIC_0015274.jpg', 'ISIC_0015353.jpg', 'ISIC_0015566.jpg', 'ISIC_0015625.jpg', 'ISIC_0015636.jpg', 'ISIC_0016009.jpg', 'ISIC_0016015.jpg', 'ISIC_0016014.jpg', 'ISIC_0007557.jpg', 'ISIC_0007693.jpg', 'ISIC_0009297.jpg', 'ISIC_0009891.jpg', 'ISIC_0009880.jpg', 'ISIC_0009918.jpg', 'ISIC_0009915.jpg', 'ISIC_0009899.jpg', 'ISIC_0009934.jpg', 'ISIC_0009940.jpg', 'ISIC_0009938.jpg', 'ISIC_0009946.jpg', 'ISIC_0009950.jpg', 'ISIC_0009948.jpg', 'ISIC_0009967.jpg', 'ISIC_0009974.jpg', 'ISIC_0009993.jpg', 'ISIC_0009992.jpg', 'ISIC_0010003.jpg', 'ISIC_0010013.jpg', 'ISIC_0010025.jpg', 'ISIC_0010021.jpg', 'ISIC_0010037.jpg', 'ISIC_0010066.jpg', 'ISIC_0010071.jpg', 'ISIC_0010175.jpg', 'ISIC_0010189.jpg', 'ISIC_0010204.jpg', 'ISIC_0010238.jpg', 'ISIC_0010235.jpg', 'ISIC_0010227.jpg', 'ISIC_0010237.jpg', 'ISIC_0010241.jpg', 'ISIC_0010239.jpg', 'ISIC_0010265.jpg', 'ISIC_0010342.jpg', 'ISIC_0010361.jpg', 'ISIC_0010349.jpg', 'ISIC_0010358.jpg', 'ISIC_0010360.jpg', 'ISIC_0010364.jpg', 'ISIC_0010487.jpg', 'ISIC_0010572.jpg', 'ISIC_0011099.jpg', 'ISIC_0011088.jpg', 'ISIC_0011110.jpg', 'ISIC_0011137.jpg', 'ISIC_0011158.jpg', 'ISIC_0011161.jpg', 'ISIC_0011167.jpg', 'ISIC_0011223.jpg', 'ISIC_0011228.jpg', 'ISIC_0011324.jpg', 'ISIC_0011317.jpg', 'ISIC_0011322.jpg', 'ISIC_0011326.jpg', 'ISIC_0011329.jpg', 'ISIC_0011328.jpg', 'ISIC_0011356.jpg', 'ISIC_0012092.jpg', 'ISIC_0012221.jpg', 'ISIC_0012233.jpg', 'ISIC_0012290.jpg', 'ISIC_0012318.jpg', 'ISIC_0012357.jpg', 'ISIC_0012400.jpg', 'ISIC_0012413.jpg', 'ISIC_0012658.jpg', 'ISIC_0012721.jpg', 'ISIC_0012768.jpg', 'ISIC_0012804.jpg', 'ISIC_0012989.jpg', 'ISIC_0012990.jpg', 'ISIC_0012988.jpg', 'ISIC_0013015.jpg', 'ISIC_0000007.jpg', 'ISIC_0000024.jpg', 'ISIC_0000045.jpg', 'ISIC_0000043.jpg', 'ISIC_0000052.jpg', 'ISIC_0000071.jpg', 'ISIC_0000092.jpg', 'ISIC_0000087.jpg', 'ISIC_0000117.jpg', 'ISIC_0000133.jpg', 'ISIC_0000146.jpg', 'ISIC_0000160.jpg', 'ISIC_0000152.jpg', 'ISIC_0000203.jpg', 'ISIC_0000222.jpg', 'ISIC_0000249.jpg', 'ISIC_0000281.jpg', 'ISIC_0000293.jpg', 'ISIC_0000295.jpg', 'ISIC_0000331.jpg', 'ISIC_0000358.jpg', 'ISIC_0000407.jpg', 'ISIC_0000396.jpg', 'ISIC_0000455.jpg', 'ISIC_0000442.jpg', 'ISIC_0000453.jpg', 'ISIC_0000507.jpg', 'ISIC_0000531.jpg', 'ISIC_0000546.jpg', 'ISIC_0001105.jpg', 'ISIC_0000552.jpg', 'ISIC_0001128.jpg', 'ISIC_0001163.jpg', 'ISIC_0001133.jpg', 'ISIC_0001385.jpg']

training_attribute_pigment_network = ['ISIC_0013275.jpg', 'ISIC_0013277.jpg', 'ISIC_0013302.jpg', 'ISIC_0013291.jpg', 'ISIC_0013288.jpg', 'ISIC_0013287.jpg', 'ISIC_0013274.jpg', 'ISIC_0013294.jpg', 'ISIC_0013304.jpg', 'ISIC_0013320.jpg', 'ISIC_0013314.jpg', 'ISIC_0013326.jpg', 'ISIC_0013330.jpg', 'ISIC_0013329.jpg', 'ISIC_0013356.jpg', 'ISIC_0013359.jpg', 'ISIC_0013374.jpg', 'ISIC_0013378.jpg', 'ISIC_0013385.jpg', 'ISIC_0013414.jpg', 'ISIC_0013411.jpg', 'ISIC_0013398.jpg', 'ISIC_0013417.jpg', 'ISIC_0013428.jpg', 'ISIC_0013432.jpg', 'ISIC_0013455.jpg', 'ISIC_0013434.jpg', 'ISIC_0013473.jpg', 'ISIC_0013488.jpg', 'ISIC_0013491.jpg', 'ISIC_0013487.jpg', 'ISIC_0013490.jpg', 'ISIC_0013480.jpg', 'ISIC_0013489.jpg', 'ISIC_0013498.jpg', 'ISIC_0013495.jpg', 'ISIC_0013494.jpg', 'ISIC_0013492.jpg', 'ISIC_0013499.jpg', 'ISIC_0013493.jpg', 'ISIC_0013500.jpg', 'ISIC_0013523.jpg', 'ISIC_0013526.jpg', 'ISIC_0013529.jpg', 'ISIC_0013525.jpg', 'ISIC_0013517.jpg', 'ISIC_0013549.jpg', 'ISIC_0013559.jpg', 'ISIC_0013562.jpg', 'ISIC_0013554.jpg', 'ISIC_0013553.jpg', 'ISIC_0013568.jpg', 'ISIC_0013561.jpg', 'ISIC_0013567.jpg', 'ISIC_0013573.jpg', 'ISIC_0013577.jpg', 'ISIC_0013579.jpg', 'ISIC_0013606.jpg', 'ISIC_0013597.jpg', 'ISIC_0013603.jpg', 'ISIC_0013596.jpg', 'ISIC_0013595.jpg', 'ISIC_0013594.jpg', 'ISIC_0013601.jpg', 'ISIC_0013634.jpg', 'ISIC_0013615.jpg', 'ISIC_0013618.jpg', 'ISIC_0013626.jpg', 'ISIC_0013651.jpg', 'ISIC_0013663.jpg', 'ISIC_0013639.jpg', 'ISIC_0013664.jpg', 'ISIC_0013656.jpg', 'ISIC_0013667.jpg', 'ISIC_0013674.jpg', 'ISIC_0013672.jpg', 'ISIC_0013670.jpg', 'ISIC_0013702.jpg', 'ISIC_0013695.jpg', 'ISIC_0013690.jpg', 'ISIC_0013689.jpg', 'ISIC_0013696.jpg', 'ISIC_0013733.jpg', 'ISIC_0013736.jpg', 'ISIC_0013719.jpg', 'ISIC_0013737.jpg', 'ISIC_0013740.jpg', 'ISIC_0013758.jpg', 'ISIC_0013772.jpg', 'ISIC_0013766.jpg', 'ISIC_0013748.jpg', 'ISIC_0013749.jpg', 'ISIC_0013747.jpg', 'ISIC_0013775.jpg', 'ISIC_0013778.jpg', 'ISIC_0013792.jpg', 'ISIC_0013795.jpg', 'ISIC_0013789.jpg', 'ISIC_0013783.jpg', 'ISIC_0013794.jpg', 'ISIC_0013805.jpg', 'ISIC_0013807.jpg', 'ISIC_0013806.jpg', 'ISIC_0013798.jpg', 'ISIC_0013801.jpg', 'ISIC_0013804.jpg', 'ISIC_0013809.jpg', 'ISIC_0013831.jpg', 'ISIC_0013830.jpg', 'ISIC_0013808.jpg', 'ISIC_0013832.jpg', 'ISIC_0013833.jpg', 'ISIC_0013861.jpg', 'ISIC_0013862.jpg', 'ISIC_0013865.jpg', 'ISIC_0013874.jpg', 'ISIC_0013891.jpg', 'ISIC_0013886.jpg', 'ISIC_0013921.jpg', 'ISIC_0013908.jpg', 'ISIC_0013896.jpg', 'ISIC_0013922.jpg', 'ISIC_0013917.jpg', 'ISIC_0013962.jpg', 'ISIC_0013929.jpg', 'ISIC_0013942.jpg', 'ISIC_0013936.jpg', 'ISIC_0013946.jpg', 'ISIC_0013953.jpg', 'ISIC_0013948.jpg', 'ISIC_0013969.jpg', 'ISIC_0013972.jpg', 'ISIC_0013970.jpg', 'ISIC_0013987.jpg', 'ISIC_0013984.jpg', 'ISIC_0014013.jpg', 'ISIC_0013983.jpg', 'ISIC_0013996.jpg', 'ISIC_0013995.jpg', 'ISIC_0013986.jpg', 'ISIC_0013997.jpg', 'ISIC_0014059.jpg', 'ISIC_0014027.jpg', 'ISIC_0014049.jpg', 'ISIC_0014073.jpg', 'ISIC_0014080.jpg', 'ISIC_0014069.jpg', 'ISIC_0014066.jpg', 'ISIC_0014077.jpg', 'ISIC_0014089.jpg', 'ISIC_0014081.jpg', 'ISIC_0014062.jpg', 'ISIC_0014093.jpg', 'ISIC_0014110.jpg', 'ISIC_0014090.jpg', 'ISIC_0014103.jpg', 'ISIC_0014094.jpg', 'ISIC_0014108.jpg', 'ISIC_0014117.jpg', 'ISIC_0014092.jpg', 'ISIC_0014150.jpg', 'ISIC_0014129.jpg', 'ISIC_0014156.jpg', 'ISIC_0014131.jpg', 'ISIC_0014133.jpg', 'ISIC_0014151.jpg', 'ISIC_0014164.jpg', 'ISIC_0014174.jpg', 'ISIC_0014163.jpg', 'ISIC_0014157.jpg', 'ISIC_0014178.jpg', 'ISIC_0014211.jpg', 'ISIC_0014189.jpg', 'ISIC_0014183.jpg', 'ISIC_0014181.jpg', 'ISIC_0014197.jpg', 'ISIC_0014217.jpg', 'ISIC_0014229.jpg', 'ISIC_0014249.jpg', 'ISIC_0014233.jpg', 'ISIC_0014221.jpg', 'ISIC_0014238.jpg', 'ISIC_0014288.jpg', 'ISIC_0014289.jpg', 'ISIC_0014255.jpg', 'ISIC_0014286.jpg', 'ISIC_0014263.jpg', 'ISIC_0014291.jpg', 'ISIC_0014325.jpg', 'ISIC_0014317.jpg', 'ISIC_0014327.jpg', 'ISIC_0014302.jpg', 'ISIC_0014308.jpg', 'ISIC_0014324.jpg', 'ISIC_0014360.jpg', 'ISIC_0014346.jpg', 'ISIC_0014353.jpg', 'ISIC_0014331.jpg', 'ISIC_0014347.jpg', 'ISIC_0014349.jpg', 'ISIC_0014372.jpg', 'ISIC_0014394.jpg', 'ISIC_0014361.jpg', 'ISIC_0014385.jpg', 'ISIC_0014365.jpg', 'ISIC_0014366.jpg', 'ISIC_0014428.jpg', 'ISIC_0014433.jpg', 'ISIC_0014431.jpg', 'ISIC_0014434.jpg', 'ISIC_0014430.jpg', 'ISIC_0014422.jpg', 'ISIC_0014473.jpg', 'ISIC_0014458.jpg', 'ISIC_0014475.jpg', 'ISIC_0014441.jpg', 'ISIC_0014453.jpg', 'ISIC_0014490.jpg', 'ISIC_0014486.jpg', 'ISIC_0014502.jpg', 'ISIC_0014478.jpg', 'ISIC_0014476.jpg', 'ISIC_0014480.jpg', 'ISIC_0014489.jpg', 'ISIC_0014498.jpg', 'ISIC_0014501.jpg', 'ISIC_0014527.jpg', 'ISIC_0014522.jpg', 'ISIC_0014513.jpg', 'ISIC_0014506.jpg', 'ISIC_0014518.jpg', 'ISIC_0014542.jpg', 'ISIC_0014545.jpg', 'ISIC_0014529.jpg', 'ISIC_0014548.jpg', 'ISIC_0014541.jpg', 'ISIC_0014547.jpg', 'ISIC_0014576.jpg', 'ISIC_0014567.jpg', 'ISIC_0014558.jpg', 'ISIC_0014559.jpg', 'ISIC_0014585.jpg', 'ISIC_0014586.jpg', 'ISIC_0014603.jpg', 'ISIC_0014661.jpg', 'ISIC_0014658.jpg', 'ISIC_0014665.jpg', 'ISIC_0014677.jpg', 'ISIC_0014682.jpg', 'ISIC_0014685.jpg', 'ISIC_0014687.jpg', 'ISIC_0014692.jpg', 'ISIC_0014688.jpg', 'ISIC_0014693.jpg', 'ISIC_0014702.jpg', 'ISIC_0014703.jpg', 'ISIC_0014696.jpg', 'ISIC_0014694.jpg', 'ISIC_0014697.jpg', 'ISIC_0014720.jpg', 'ISIC_0014707.jpg', 'ISIC_0014716.jpg', 'ISIC_0014715.jpg', 'ISIC_0014713.jpg', 'ISIC_0014708.jpg', 'ISIC_0014728.jpg', 'ISIC_0014722.jpg', 'ISIC_0014726.jpg', 'ISIC_0014724.jpg', 'ISIC_0014725.jpg', 'ISIC_0014740.jpg', 'ISIC_0014731.jpg', 'ISIC_0014735.jpg', 'ISIC_0014739.jpg', 'ISIC_0014730.jpg', 'ISIC_0014743.jpg', 'ISIC_0014742.jpg', 'ISIC_0014745.jpg', 'ISIC_0014753.jpg', 'ISIC_0014754.jpg', 'ISIC_0014749.jpg', 'ISIC_0014755.jpg', 'ISIC_0014748.jpg', 'ISIC_0014762.jpg', 'ISIC_0014763.jpg', 'ISIC_0014760.jpg', 'ISIC_0014768.jpg', 'ISIC_0014775.jpg', 'ISIC_0014771.jpg', 'ISIC_0014778.jpg', 'ISIC_0014772.jpg', 'ISIC_0014773.jpg', 'ISIC_0014780.jpg', 'ISIC_0014785.jpg', 'ISIC_0014794.jpg', 'ISIC_0014795.jpg', 'ISIC_0014788.jpg', 'ISIC_0014791.jpg', 'ISIC_0014803.jpg', 'ISIC_0014796.jpg', 'ISIC_0014800.jpg', 'ISIC_0014802.jpg', 'ISIC_0014798.jpg', 'ISIC_0014807.jpg', 'ISIC_0014805.jpg', 'ISIC_0014809.jpg', 'ISIC_0014814.jpg', 'ISIC_0014808.jpg', 'ISIC_0014826.jpg', 'ISIC_0014825.jpg', 'ISIC_0014818.jpg', 'ISIC_0014823.jpg', 'ISIC_0014821.jpg', 'ISIC_0014820.jpg', 'ISIC_0014815.jpg', 'ISIC_0014830.jpg', 'ISIC_0014829.jpg', 'ISIC_0014834.jpg', 'ISIC_0014832.jpg', 'ISIC_0014831.jpg', 'ISIC_0014843.jpg', 'ISIC_0014836.jpg', 'ISIC_0014835.jpg', 'ISIC_0014838.jpg', 'ISIC_0014844.jpg', 'ISIC_0014848.jpg', 'ISIC_0014845.jpg', 'ISIC_0014846.jpg', 'ISIC_0014854.jpg', 'ISIC_0014855.jpg', 'ISIC_0014850.jpg', 'ISIC_0014851.jpg', 'ISIC_0014868.jpg', 'ISIC_0014862.jpg', 'ISIC_0014863.jpg', 'ISIC_0014860.jpg', 'ISIC_0014857.jpg', 'ISIC_0014869.jpg', 'ISIC_0014876.jpg', 'ISIC_0014890.jpg', 'ISIC_0014891.jpg', 'ISIC_0014907.jpg', 'ISIC_0014908.jpg', 'ISIC_0014904.jpg', 'ISIC_0014903.jpg', 'ISIC_0014898.jpg', 'ISIC_0014901.jpg', 'ISIC_0014910.jpg', 'ISIC_0014913.jpg', 'ISIC_0014919.jpg', 'ISIC_0014912.jpg', 'ISIC_0014911.jpg', 'ISIC_0014915.jpg', 'ISIC_0014921.jpg', 'ISIC_0014922.jpg', 'ISIC_0014923.jpg', 'ISIC_0014920.jpg', 'ISIC_0014925.jpg', 'ISIC_0014929.jpg', 'ISIC_0014927.jpg', 'ISIC_0014932.jpg', 'ISIC_0014931.jpg', 'ISIC_0014930.jpg', 'ISIC_0014938.jpg', 'ISIC_0014940.jpg', 'ISIC_0014936.jpg', 'ISIC_0014933.jpg', 'ISIC_0014942.jpg', 'ISIC_0014943.jpg', 'ISIC_0014944.jpg', 'ISIC_0014945.jpg', 'ISIC_0014946.jpg', 'ISIC_0014941.jpg', 'ISIC_0014949.jpg', 'ISIC_0014951.jpg', 'ISIC_0014947.jpg', 'ISIC_0014957.jpg', 'ISIC_0014952.jpg', 'ISIC_0014955.jpg', 'ISIC_0014956.jpg', 'ISIC_0014966.jpg', 'ISIC_0014964.jpg', 'ISIC_0014962.jpg', 'ISIC_0014963.jpg', 'ISIC_0014958.jpg', 'ISIC_0014979.jpg', 'ISIC_0014968.jpg', 'ISIC_0014974.jpg', 'ISIC_0014973.jpg', 'ISIC_0014989.jpg', 'ISIC_0014992.jpg', 'ISIC_0014987.jpg', 'ISIC_0014982.jpg', 'ISIC_0014983.jpg', 'ISIC_0015005.jpg', 'ISIC_0014998.jpg', 'ISIC_0015019.jpg', 'ISIC_0015015.jpg', 'ISIC_0015011.jpg', 'ISIC_0015013.jpg', 'ISIC_0015016.jpg', 'ISIC_0015023.jpg', 'ISIC_0015020.jpg', 'ISIC_0015034.jpg', 'ISIC_0015041.jpg', 'ISIC_0015037.jpg', 'ISIC_0015046.jpg', 'ISIC_0015051.jpg', 'ISIC_0015050.jpg', 'ISIC_0015056.jpg', 'ISIC_0015079.jpg', 'ISIC_0015071.jpg', 'ISIC_0015078.jpg', 'ISIC_0015082.jpg', 'ISIC_0015064.jpg', 'ISIC_0015113.jpg', 'ISIC_0015109.jpg', 'ISIC_0015110.jpg', 'ISIC_0015089.jpg', 'ISIC_0015118.jpg', 'ISIC_0015127.jpg', 'ISIC_0015115.jpg', 'ISIC_0015125.jpg', 'ISIC_0015124.jpg', 'ISIC_0015130.jpg', 'ISIC_0015139.jpg', 'ISIC_0015136.jpg', 'ISIC_0015133.jpg', 'ISIC_0015132.jpg', 'ISIC_0015142.jpg', 'ISIC_0015150.jpg', 'ISIC_0015152.jpg', 'ISIC_0015146.jpg', 'ISIC_0015149.jpg', 'ISIC_0015144.jpg', 'ISIC_0015158.jpg', 'ISIC_0015157.jpg', 'ISIC_0015156.jpg', 'ISIC_0015163.jpg', 'ISIC_0015173.jpg', 'ISIC_0015166.jpg', 'ISIC_0015174.jpg', 'ISIC_0015171.jpg', 'ISIC_0015180.jpg', 'ISIC_0015179.jpg', 'ISIC_0015176.jpg', 'ISIC_0015184.jpg', 'ISIC_0015175.jpg', 'ISIC_0015190.jpg', 'ISIC_0015193.jpg', 'ISIC_0015185.jpg', 'ISIC_0015202.jpg', 'ISIC_0015204.jpg', 'ISIC_0015200.jpg', 'ISIC_0015201.jpg', 'ISIC_0015203.jpg', 'ISIC_0015208.jpg', 'ISIC_0015216.jpg', 'ISIC_0015211.jpg', 'ISIC_0015212.jpg', 'ISIC_0015215.jpg', 'ISIC_0015206.jpg', 'ISIC_0015218.jpg', 'ISIC_0015220.jpg', 'ISIC_0015224.jpg', 'ISIC_0015229.jpg', 'ISIC_0015219.jpg', 'ISIC_0015232.jpg', 'ISIC_0015243.jpg', 'ISIC_0015241.jpg', 'ISIC_0015237.jpg', 'ISIC_0015244.jpg', 'ISIC_0015251.jpg', 'ISIC_0015256.jpg', 'ISIC_0015250.jpg', 'ISIC_0015258.jpg', 'ISIC_0015254.jpg', 'ISIC_0015255.jpg', 'ISIC_0015260.jpg', 'ISIC_0015270.jpg', 'ISIC_0015283.jpg', 'ISIC_0015293.jpg', 'ISIC_0015279.jpg', 'ISIC_0015298.jpg', 'ISIC_0015312.jpg', 'ISIC_0015311.jpg', 'ISIC_0015330.jpg', 'ISIC_0015313.jpg', 'ISIC_0015368.jpg', 'ISIC_0015360.jpg', 'ISIC_0015383.jpg', 'ISIC_0015390.jpg', 'ISIC_0015401.jpg', 'ISIC_0015411.jpg', 'ISIC_0015412.jpg', 'ISIC_0015418.jpg', 'ISIC_0015417.jpg', 'ISIC_0015447.jpg', 'ISIC_0015419.jpg', 'ISIC_0015455.jpg', 'ISIC_0015468.jpg', 'ISIC_0015466.jpg', 'ISIC_0015476.jpg', 'ISIC_0015482.jpg', 'ISIC_0015464.jpg', 'ISIC_0015544.jpg', 'ISIC_0015537.jpg', 'ISIC_0015526.jpg', 'ISIC_0015566.jpg', 'ISIC_0015568.jpg', 'ISIC_0015593.jpg', 'ISIC_0015582.jpg', 'ISIC_0015614.jpg', 'ISIC_0015617.jpg', 'ISIC_0015636.jpg', 'ISIC_0015638.jpg', 'ISIC_0015631.jpg', 'ISIC_0015645.jpg', 'ISIC_0015942.jpg', 'ISIC_0015937.jpg', 'ISIC_0015936.jpg', 'ISIC_0015940.jpg', 'ISIC_0015946.jpg', 'ISIC_0015941.jpg', 'ISIC_0015945.jpg', 'ISIC_0015939.jpg', 'ISIC_0015948.jpg', 'ISIC_0015953.jpg', 'ISIC_0015949.jpg', 'ISIC_0015950.jpg', 'ISIC_0015956.jpg', 'ISIC_0015962.jpg', 'ISIC_0015966.jpg', 'ISIC_0015964.jpg', 'ISIC_0015967.jpg', 'ISIC_0015960.jpg', 'ISIC_0015963.jpg', 'ISIC_0015961.jpg', 'ISIC_0015971.jpg', 'ISIC_0015969.jpg', 'ISIC_0015968.jpg', 'ISIC_0015982.jpg', 'ISIC_0015975.jpg', 'ISIC_0015980.jpg', 'ISIC_0015981.jpg', 'ISIC_0015995.jpg', 'ISIC_0015998.jpg', 'ISIC_0015996.jpg', 'ISIC_0015986.jpg', 'ISIC_0015989.jpg', 'ISIC_0015997.jpg', 'ISIC_0015994.jpg', 'ISIC_0015990.jpg', 'ISIC_0015999.jpg', 'ISIC_0015987.jpg', 'ISIC_0016006.jpg', 'ISIC_0016012.jpg', 'ISIC_0016009.jpg', 'ISIC_0016003.jpg', 'ISIC_0016005.jpg', 'ISIC_0016008.jpg', 'ISIC_0016004.jpg', 'ISIC_0016022.jpg', 'ISIC_0016023.jpg', 'ISIC_0016016.jpg', 'ISIC_0016013.jpg', 'ISIC_0016015.jpg', 'ISIC_0016024.jpg', 'ISIC_0016019.jpg', 'ISIC_0016026.jpg', 'ISIC_0016025.jpg', 'ISIC_0016036.jpg', 'ISIC_0016034.jpg', 'ISIC_0016030.jpg', 'ISIC_0016037.jpg', 'ISIC_0016033.jpg', 'ISIC_0016027.jpg', 'ISIC_0016035.jpg', 'ISIC_0016038.jpg', 'ISIC_0016029.jpg', 'ISIC_0016045.jpg', 'ISIC_0016046.jpg', 'ISIC_0016050.jpg', 'ISIC_0016040.jpg', 'ISIC_0016048.jpg', 'ISIC_0016057.jpg', 'ISIC_0016056.jpg', 'ISIC_0016062.jpg', 'ISIC_0016053.jpg', 'ISIC_0016059.jpg', 'ISIC_0016066.jpg', 'ISIC_0016065.jpg', 'ISIC_0016064.jpg', 'ISIC_0016063.jpg', 'ISIC_0016070.jpg', 'ISIC_0016069.jpg', 'ISIC_0016072.jpg', 'ISIC_0016071.jpg', 'ISIC_0016068.jpg', 'ISIC_0007141.jpg', 'ISIC_0007788.jpg', 'ISIC_0007796.jpg', 'ISIC_0008406.jpg', 'ISIC_0008507.jpg', 'ISIC_0008807.jpg', 'ISIC_0008879.jpg', 'ISIC_0008998.jpg', 'ISIC_0009297.jpg', 'ISIC_0009160.jpg', 'ISIC_0009201.jpg', 'ISIC_0009344.jpg', 'ISIC_0009430.jpg', 'ISIC_0009078.jpg', 'ISIC_0009165.jpg', 'ISIC_0009869.jpg', 'ISIC_0009874.jpg', 'ISIC_0009800.jpg', 'ISIC_0009758.jpg', 'ISIC_0009860.jpg', 'ISIC_0009896.jpg', 'ISIC_0009895.jpg', 'ISIC_0009880.jpg', 'ISIC_0009888.jpg', 'ISIC_0009881.jpg', 'ISIC_0009883.jpg', 'ISIC_0009875.jpg', 'ISIC_0009885.jpg', 'ISIC_0009879.jpg', 'ISIC_0009884.jpg', 'ISIC_0009910.jpg', 'ISIC_0009902.jpg', 'ISIC_0009898.jpg', 'ISIC_0009915.jpg', 'ISIC_0009901.jpg', 'ISIC_0009914.jpg', 'ISIC_0009909.jpg', 'ISIC_0009897.jpg', 'ISIC_0009931.jpg', 'ISIC_0009925.jpg', 'ISIC_0009923.jpg', 'ISIC_0009930.jpg', 'ISIC_0009932.jpg', 'ISIC_0009936.jpg', 'ISIC_0009940.jpg', 'ISIC_0009935.jpg', 'ISIC_0009944.jpg', 'ISIC_0009941.jpg', 'ISIC_0009938.jpg', 'ISIC_0009947.jpg', 'ISIC_0009946.jpg', 'ISIC_0009939.jpg', 'ISIC_0009943.jpg', 'ISIC_0009961.jpg', 'ISIC_0009949.jpg', 'ISIC_0009956.jpg', 'ISIC_0009953.jpg', 'ISIC_0009958.jpg', 'ISIC_0009954.jpg', 'ISIC_0009962.jpg', 'ISIC_0009948.jpg', 'ISIC_0009960.jpg', 'ISIC_0009975.jpg', 'ISIC_0009970.jpg', 'ISIC_0009972.jpg', 'ISIC_0009977.jpg', 'ISIC_0009969.jpg', 'ISIC_0009973.jpg', 'ISIC_0009998.jpg', 'ISIC_0009993.jpg', 'ISIC_0009990.jpg', 'ISIC_0009994.jpg', 'ISIC_0009980.jpg', 'ISIC_0009991.jpg', 'ISIC_0009981.jpg', 'ISIC_0009992.jpg', 'ISIC_0009995.jpg', 'ISIC_0009987.jpg', 'ISIC_0009979.jpg', 'ISIC_0010006.jpg', 'ISIC_0010003.jpg', 'ISIC_0010015.jpg', 'ISIC_0010000.jpg', 'ISIC_0010013.jpg', 'ISIC_0010014.jpg', 'ISIC_0010017.jpg', 'ISIC_0010005.jpg', 'ISIC_0010016.jpg', 'ISIC_0010019.jpg', 'ISIC_0010028.jpg', 'ISIC_0010023.jpg', 'ISIC_0010024.jpg', 'ISIC_0010020.jpg', 'ISIC_0010018.jpg', 'ISIC_0010022.jpg', 'ISIC_0010056.jpg', 'ISIC_0010055.jpg', 'ISIC_0010053.jpg', 'ISIC_0010057.jpg', 'ISIC_0010041.jpg', 'ISIC_0010066.jpg', 'ISIC_0010063.jpg', 'ISIC_0010070.jpg', 'ISIC_0010058.jpg', 'ISIC_0010071.jpg', 'ISIC_0010062.jpg', 'ISIC_0010077.jpg', 'ISIC_0010100.jpg', 'ISIC_0010089.jpg', 'ISIC_0010078.jpg', 'ISIC_0010092.jpg', 'ISIC_0010074.jpg', 'ISIC_0010086.jpg', 'ISIC_0010093.jpg', 'ISIC_0010075.jpg', 'ISIC_0010176.jpg', 'ISIC_0010168.jpg', 'ISIC_0010180.jpg', 'ISIC_0010174.jpg', 'ISIC_0010171.jpg', 'ISIC_0010173.jpg', 'ISIC_0010177.jpg', 'ISIC_0010104.jpg', 'ISIC_0010105.jpg', 'ISIC_0010178.jpg', 'ISIC_0010186.jpg', 'ISIC_0010191.jpg', 'ISIC_0010183.jpg', 'ISIC_0010194.jpg', 'ISIC_0010182.jpg', 'ISIC_0010187.jpg', 'ISIC_0010192.jpg', 'ISIC_0010201.jpg', 'ISIC_0010190.jpg', 'ISIC_0010220.jpg', 'ISIC_0010213.jpg', 'ISIC_0010204.jpg', 'ISIC_0010219.jpg', 'ISIC_0010205.jpg', 'ISIC_0010222.jpg', 'ISIC_0010215.jpg', 'ISIC_0010225.jpg', 'ISIC_0010202.jpg', 'ISIC_0010212.jpg', 'ISIC_0010238.jpg', 'ISIC_0010229.jpg', 'ISIC_0010236.jpg', 'ISIC_0010228.jpg', 'ISIC_0010256.jpg', 'ISIC_0010249.jpg', 'ISIC_0010247.jpg', 'ISIC_0010248.jpg', 'ISIC_0010244.jpg', 'ISIC_0010239.jpg', 'ISIC_0010252.jpg', 'ISIC_0010240.jpg', 'ISIC_0010318.jpg', 'ISIC_0010267.jpg', 'ISIC_0010321.jpg', 'ISIC_0010265.jpg', 'ISIC_0010322.jpg', 'ISIC_0010327.jpg', 'ISIC_0010329.jpg', 'ISIC_0010333.jpg', 'ISIC_0010337.jpg', 'ISIC_0010339.jpg', 'ISIC_0010336.jpg', 'ISIC_0010334.jpg', 'ISIC_0010340.jpg', 'ISIC_0010335.jpg', 'ISIC_0010342.jpg', 'ISIC_0010362.jpg', 'ISIC_0010349.jpg', 'ISIC_0010344.jpg', 'ISIC_0010350.jpg', 'ISIC_0010365.jpg', 'ISIC_0010382.jpg', 'ISIC_0010367.jpg', 'ISIC_0010374.jpg', 'ISIC_0010368.jpg', 'ISIC_0010444.jpg', 'ISIC_0010455.jpg', 'ISIC_0010443.jpg', 'ISIC_0010442.jpg', 'ISIC_0010475.jpg', 'ISIC_0010468.jpg', 'ISIC_0010474.jpg', 'ISIC_0010472.jpg', 'ISIC_0010462.jpg', 'ISIC_0010465.jpg', 'ISIC_0010488.jpg', 'ISIC_0010476.jpg', 'ISIC_0010496.jpg', 'ISIC_0010481.jpg', 'ISIC_0010492.jpg', 'ISIC_0010493.jpg', 'ISIC_0010490.jpg', 'ISIC_0010477.jpg', 'ISIC_0010480.jpg', 'ISIC_0010568.jpg', 'ISIC_0010554.jpg', 'ISIC_0010553.jpg', 'ISIC_0010558.jpg', 'ISIC_0010552.jpg', 'ISIC_0010497.jpg', 'ISIC_0010567.jpg', 'ISIC_0010570.jpg', 'ISIC_0010573.jpg', 'ISIC_0010588.jpg', 'ISIC_0010574.jpg', 'ISIC_0010589.jpg', 'ISIC_0010576.jpg', 'ISIC_0010605.jpg', 'ISIC_0010597.jpg', 'ISIC_0010599.jpg', 'ISIC_0010591.jpg', 'ISIC_0010593.jpg', 'ISIC_0010595.jpg', 'ISIC_0010602.jpg', 'ISIC_0010596.jpg', 'ISIC_0010598.jpg', 'ISIC_0010592.jpg', 'ISIC_0010606.jpg', 'ISIC_0010853.jpg', 'ISIC_0011082.jpg', 'ISIC_0011079.jpg', 'ISIC_0010857.jpg', 'ISIC_0010864.jpg', 'ISIC_0010861.jpg', 'ISIC_0010860.jpg', 'ISIC_0011084.jpg', 'ISIC_0010863.jpg', 'ISIC_0011098.jpg', 'ISIC_0011104.jpg', 'ISIC_0011092.jpg', 'ISIC_0011090.jpg', 'ISIC_0011105.jpg', 'ISIC_0011095.jpg', 'ISIC_0011102.jpg', 'ISIC_0011085.jpg', 'ISIC_0011110.jpg', 'ISIC_0011121.jpg', 'ISIC_0011124.jpg', 'ISIC_0011107.jpg', 'ISIC_0011123.jpg', 'ISIC_0011114.jpg', 'ISIC_0011135.jpg', 'ISIC_0011144.jpg', 'ISIC_0011129.jpg', 'ISIC_0011127.jpg', 'ISIC_0011131.jpg', 'ISIC_0011137.jpg', 'ISIC_0011132.jpg', 'ISIC_0011130.jpg', 'ISIC_0011139.jpg', 'ISIC_0011158.jpg', 'ISIC_0011161.jpg', 'ISIC_0011155.jpg', 'ISIC_0011164.jpg', 'ISIC_0011149.jpg', 'ISIC_0011151.jpg', 'ISIC_0011156.jpg', 'ISIC_0011163.jpg', 'ISIC_0011202.jpg', 'ISIC_0011167.jpg', 'ISIC_0011169.jpg', 'ISIC_0011170.jpg', 'ISIC_0011165.jpg', 'ISIC_0011166.jpg', 'ISIC_0011173.jpg', 'ISIC_0011177.jpg', 'ISIC_0011168.jpg', 'ISIC_0011176.jpg', 'ISIC_0011205.jpg', 'ISIC_0011219.jpg', 'ISIC_0011212.jpg', 'ISIC_0011214.jpg', 'ISIC_0011208.jpg', 'ISIC_0011204.jpg', 'ISIC_0011230.jpg', 'ISIC_0011223.jpg', 'ISIC_0011296.jpg', 'ISIC_0011297.jpg', 'ISIC_0011224.jpg', 'ISIC_0011292.jpg', 'ISIC_0011229.jpg', 'ISIC_0011295.jpg', 'ISIC_0011298.jpg', 'ISIC_0011294.jpg', 'ISIC_0011306.jpg', 'ISIC_0011300.jpg', 'ISIC_0011326.jpg', 'ISIC_0011304.jpg', 'ISIC_0011323.jpg', 'ISIC_0011338.jpg', 'ISIC_0011347.jpg', 'ISIC_0011339.jpg', 'ISIC_0011336.jpg', 'ISIC_0011330.jpg', 'ISIC_0011333.jpg', 'ISIC_0011345.jpg', 'ISIC_0011343.jpg', 'ISIC_0011334.jpg', 'ISIC_0011348.jpg', 'ISIC_0011361.jpg', 'ISIC_0011366.jpg', 'ISIC_0011362.jpg', 'ISIC_0011360.jpg', 'ISIC_0011352.jpg', 'ISIC_0011359.jpg', 'ISIC_0011363.jpg', 'ISIC_0011353.jpg', 'ISIC_0011358.jpg', 'ISIC_0011373.jpg', 'ISIC_0011385.jpg', 'ISIC_0011378.jpg', 'ISIC_0011390.jpg', 'ISIC_0011384.jpg', 'ISIC_0011382.jpg', 'ISIC_0011372.jpg', 'ISIC_0011386.jpg', 'ISIC_0011387.jpg', 'ISIC_0011367.jpg', 'ISIC_0012094.jpg', 'ISIC_0012097.jpg', 'ISIC_0012095.jpg', 'ISIC_0011402.jpg', 'ISIC_0011398.jpg', 'ISIC_0012099.jpg', 'ISIC_0012092.jpg', 'ISIC_0012105.jpg', 'ISIC_0012118.jpg', 'ISIC_0012108.jpg', 'ISIC_0012135.jpg', 'ISIC_0012107.jpg', 'ISIC_0012126.jpg', 'ISIC_0012109.jpg', 'ISIC_0012156.jpg', 'ISIC_0012137.jpg', 'ISIC_0012164.jpg', 'ISIC_0012151.jpg', 'ISIC_0012160.jpg', 'ISIC_0012148.jpg', 'ISIC_0012187.jpg', 'ISIC_0012201.jpg', 'ISIC_0012177.jpg', 'ISIC_0012205.jpg', 'ISIC_0012211.jpg', 'ISIC_0012222.jpg', 'ISIC_0012206.jpg', 'ISIC_0012230.jpg', 'ISIC_0012232.jpg', 'ISIC_0012263.jpg', 'ISIC_0012260.jpg', 'ISIC_0012259.jpg', 'ISIC_0012257.jpg', 'ISIC_0012268.jpg', 'ISIC_0012290.jpg', 'ISIC_0012288.jpg', 'ISIC_0012284.jpg', 'ISIC_0012271.jpg', 'ISIC_0012303.jpg', 'ISIC_0012316.jpg', 'ISIC_0012311.jpg', 'ISIC_0012309.jpg', 'ISIC_0012320.jpg', 'ISIC_0012333.jpg', 'ISIC_0012334.jpg', 'ISIC_0012332.jpg', 'ISIC_0012378.jpg', 'ISIC_0012376.jpg', 'ISIC_0012360.jpg', 'ISIC_0012374.jpg', 'ISIC_0012357.jpg', 'ISIC_0012351.jpg', 'ISIC_0012382.jpg', 'ISIC_0012395.jpg', 'ISIC_0012400.jpg', 'ISIC_0012453.jpg', 'ISIC_0012445.jpg', 'ISIC_0012450.jpg', 'ISIC_0012442.jpg', 'ISIC_0012493.jpg', 'ISIC_0012511.jpg', 'ISIC_0012496.jpg', 'ISIC_0012506.jpg', 'ISIC_0012510.jpg', 'ISIC_0012508.jpg', 'ISIC_0012494.jpg', 'ISIC_0012529.jpg', 'ISIC_0012520.jpg', 'ISIC_0012538.jpg', 'ISIC_0012521.jpg', 'ISIC_0012539.jpg', 'ISIC_0012656.jpg', 'ISIC_0012653.jpg', 'ISIC_0012655.jpg', 'ISIC_0012544.jpg', 'ISIC_0012549.jpg', 'ISIC_0012665.jpg', 'ISIC_0012658.jpg', 'ISIC_0012660.jpg', 'ISIC_0012659.jpg', 'ISIC_0012663.jpg', 'ISIC_0012662.jpg', 'ISIC_0012664.jpg', 'ISIC_0012680.jpg', 'ISIC_0012681.jpg', 'ISIC_0012675.jpg', 'ISIC_0012676.jpg', 'ISIC_0012679.jpg', 'ISIC_0012677.jpg', 'ISIC_0012685.jpg', 'ISIC_0012699.jpg', 'ISIC_0012684.jpg', 'ISIC_0012682.jpg', 'ISIC_0012690.jpg', 'ISIC_0012700.jpg', 'ISIC_0012683.jpg', 'ISIC_0012739.jpg', 'ISIC_0012749.jpg', 'ISIC_0012740.jpg', 'ISIC_0012741.jpg', 'ISIC_0012777.jpg', 'ISIC_0012770.jpg', 'ISIC_0012758.jpg', 'ISIC_0012756.jpg', 'ISIC_0012826.jpg', 'ISIC_0012835.jpg', 'ISIC_0012823.jpg', 'ISIC_0012810.jpg', 'ISIC_0012804.jpg', 'ISIC_0012844.jpg', 'ISIC_0012871.jpg', 'ISIC_0012873.jpg', 'ISIC_0012855.jpg', 'ISIC_0012887.jpg', 'ISIC_0012897.jpg', 'ISIC_0012901.jpg', 'ISIC_0012884.jpg', 'ISIC_0012911.jpg', 'ISIC_0012905.jpg', 'ISIC_0012907.jpg', 'ISIC_0012903.jpg', 'ISIC_0012930.jpg', 'ISIC_0012902.jpg', 'ISIC_0012961.jpg', 'ISIC_0012965.jpg', 'ISIC_0012976.jpg', 'ISIC_0012950.jpg', 'ISIC_0012944.jpg', 'ISIC_0013010.jpg', 'ISIC_0012989.jpg', 'ISIC_0012990.jpg', 'ISIC_0012988.jpg', 'ISIC_0012999.jpg', 'ISIC_0013032.jpg', 'ISIC_0013037.jpg', 'ISIC_0013023.jpg', 'ISIC_0013031.jpg', 'ISIC_0013044.jpg', 'ISIC_0013039.jpg', 'ISIC_0013047.jpg', 'ISIC_0013082.jpg', 'ISIC_0013087.jpg', 'ISIC_0013084.jpg', 'ISIC_0013070.jpg', 'ISIC_0013072.jpg', 'ISIC_0013073.jpg', 'ISIC_0013086.jpg', 'ISIC_0013079.jpg', 'ISIC_0013118.jpg', 'ISIC_0013112.jpg', 'ISIC_0013104.jpg', 'ISIC_0013094.jpg', 'ISIC_0013109.jpg', 'ISIC_0013096.jpg', 'ISIC_0013136.jpg', 'ISIC_0013126.jpg', 'ISIC_0013128.jpg', 'ISIC_0013140.jpg', 'ISIC_0013164.jpg', 'ISIC_0013147.jpg', 'ISIC_0013160.jpg', 'ISIC_0013184.jpg', 'ISIC_0013173.jpg', 'ISIC_0013187.jpg', 'ISIC_0013198.jpg', 'ISIC_0013190.jpg', 'ISIC_0013192.jpg', 'ISIC_0013204.jpg', 'ISIC_0013205.jpg', 'ISIC_0013216.jpg', 'ISIC_0013217.jpg', 'ISIC_0013220.jpg', 'ISIC_0013229.jpg', 'ISIC_0013232.jpg', 'ISIC_0013224.jpg', 'ISIC_0013230.jpg', 'ISIC_0013222.jpg', 'ISIC_0013244.jpg', 'ISIC_0013269.jpg', 'ISIC_0013249.jpg', 'ISIC_0013233.jpg', 'ISIC_0013242.jpg', 'ISIC_0013257.jpg', 'ISIC_0013248.jpg', 'ISIC_0013258.jpg', 'ISIC_0013238.jpg', 'ISIC_0000008.jpg', 'ISIC_0000003.jpg', 'ISIC_0000006.jpg', 'ISIC_0000000.jpg', 'ISIC_0000007.jpg', 'ISIC_0000001.jpg', 'ISIC_0000022.jpg', 'ISIC_0000017.jpg', 'ISIC_0000018.jpg', 'ISIC_0000015.jpg', 'ISIC_0000012.jpg', 'ISIC_0000009.jpg', 'ISIC_0000016.jpg', 'ISIC_0000020.jpg', 'ISIC_0000021.jpg', 'ISIC_0000013.jpg', 'ISIC_0000019.jpg', 'ISIC_0000029.jpg', 'ISIC_0000028.jpg', 'ISIC_0000036.jpg', 'ISIC_0000025.jpg', 'ISIC_0000032.jpg', 'ISIC_0000034.jpg', 'ISIC_0000027.jpg', 'ISIC_0000031.jpg', 'ISIC_0000039.jpg', 'ISIC_0000044.jpg', 'ISIC_0000047.jpg', 'ISIC_0000048.jpg', 'ISIC_0000046.jpg', 'ISIC_0000037.jpg', 'ISIC_0000043.jpg', 'ISIC_0000038.jpg', 'ISIC_0000049.jpg', 'ISIC_0000042.jpg', 'ISIC_0000059.jpg', 'ISIC_0000052.jpg', 'ISIC_0000058.jpg', 'ISIC_0000056.jpg', 'ISIC_0000057.jpg', 'ISIC_0000054.jpg', 'ISIC_0000051.jpg', 'ISIC_0000062.jpg', 'ISIC_0000055.jpg', 'ISIC_0000053.jpg', 'ISIC_0000060.jpg', 'ISIC_0000061.jpg', 'ISIC_0000063.jpg', 'ISIC_0000050.jpg', 'ISIC_0000072.jpg', 'ISIC_0000078.jpg', 'ISIC_0000075.jpg', 'ISIC_0000074.jpg', 'ISIC_0000067.jpg', 'ISIC_0000066.jpg', 'ISIC_0000071.jpg', 'ISIC_0000077.jpg', 'ISIC_0000064.jpg', 'ISIC_0000073.jpg', 'ISIC_0000068.jpg', 'ISIC_0000069.jpg', 'ISIC_0000065.jpg', 'ISIC_0000094.jpg', 'ISIC_0000079.jpg', 'ISIC_0000085.jpg', 'ISIC_0000088.jpg', 'ISIC_0000091.jpg', 'ISIC_0000082.jpg', 'ISIC_0000089.jpg', 'ISIC_0000086.jpg', 'ISIC_0000087.jpg', 'ISIC_0000080.jpg', 'ISIC_0000093.jpg', 'ISIC_0000107.jpg', 'ISIC_0000103.jpg', 'ISIC_0000096.jpg', 'ISIC_0000102.jpg', 'ISIC_0000105.jpg', 'ISIC_0000108.jpg', 'ISIC_0000095.jpg', 'ISIC_0000100.jpg', 'ISIC_0000097.jpg', 'ISIC_0000117.jpg', 'ISIC_0000116.jpg', 'ISIC_0000112.jpg', 'ISIC_0000120.jpg', 'ISIC_0000109.jpg', 'ISIC_0000119.jpg', 'ISIC_0000111.jpg', 'ISIC_0000136.jpg', 'ISIC_0000135.jpg', 'ISIC_0000126.jpg', 'ISIC_0000124.jpg', 'ISIC_0000133.jpg', 'ISIC_0000127.jpg', 'ISIC_0000129.jpg', 'ISIC_0000134.jpg', 'ISIC_0000128.jpg', 'ISIC_0000139.jpg', 'ISIC_0000140.jpg', 'ISIC_0000137.jpg', 'ISIC_0000143.jpg', 'ISIC_0000149.jpg', 'ISIC_0000142.jpg', 'ISIC_0000138.jpg', 'ISIC_0000141.jpg', 'ISIC_0000145.jpg', 'ISIC_0000150.jpg', 'ISIC_0000148.jpg', 'ISIC_0000157.jpg', 'ISIC_0000160.jpg', 'ISIC_0000162.jpg', 'ISIC_0000159.jpg', 'ISIC_0000153.jpg', 'ISIC_0000163.jpg', 'ISIC_0000151.jpg', 'ISIC_0000161.jpg', 'ISIC_0000155.jpg', 'ISIC_0000154.jpg', 'ISIC_0000152.jpg', 'ISIC_0000164.jpg', 'ISIC_0000156.jpg', 'ISIC_0000175.jpg', 'ISIC_0000166.jpg', 'ISIC_0000165.jpg', 'ISIC_0000171.jpg', 'ISIC_0000167.jpg', 'ISIC_0000172.jpg', 'ISIC_0000173.jpg', 'ISIC_0000176.jpg', 'ISIC_0000170.jpg', 'ISIC_0000184.jpg', 'ISIC_0000183.jpg', 'ISIC_0000196.jpg', 'ISIC_0000182.jpg', 'ISIC_0000190.jpg', 'ISIC_0000191.jpg', 'ISIC_0000197.jpg', 'ISIC_0000206.jpg', 'ISIC_0000203.jpg', 'ISIC_0000201.jpg', 'ISIC_0000199.jpg', 'ISIC_0000209.jpg', 'ISIC_0000200.jpg', 'ISIC_0000204.jpg', 'ISIC_0000202.jpg', 'ISIC_0000207.jpg', 'ISIC_0000205.jpg', 'ISIC_0000215.jpg', 'ISIC_0000222.jpg', 'ISIC_0000219.jpg', 'ISIC_0000218.jpg', 'ISIC_0000211.jpg', 'ISIC_0000220.jpg', 'ISIC_0000212.jpg', 'ISIC_0000221.jpg', 'ISIC_0000217.jpg', 'ISIC_0000213.jpg', 'ISIC_0000214.jpg', 'ISIC_0000216.jpg', 'ISIC_0000236.jpg', 'ISIC_0000234.jpg', 'ISIC_0000229.jpg', 'ISIC_0000224.jpg', 'ISIC_0000225.jpg', 'ISIC_0000230.jpg', 'ISIC_0000232.jpg', 'ISIC_0000223.jpg', 'ISIC_0000231.jpg', 'ISIC_0000226.jpg', 'ISIC_0000233.jpg', 'ISIC_0000227.jpg', 'ISIC_0000228.jpg', 'ISIC_0000250.jpg', 'ISIC_0000246.jpg', 'ISIC_0000242.jpg', 'ISIC_0000253.jpg', 'ISIC_0000252.jpg', 'ISIC_0000239.jpg', 'ISIC_0000240.jpg', 'ISIC_0000243.jpg', 'ISIC_0000247.jpg', 'ISIC_0000245.jpg', 'ISIC_0000251.jpg', 'ISIC_0000244.jpg', 'ISIC_0000263.jpg', 'ISIC_0000254.jpg', 'ISIC_0000260.jpg', 'ISIC_0000261.jpg', 'ISIC_0000269.jpg', 'ISIC_0000268.jpg', 'ISIC_0000259.jpg', 'ISIC_0000258.jpg', 'ISIC_0000264.jpg', 'ISIC_0000255.jpg', 'ISIC_0000262.jpg', 'ISIC_0000256.jpg', 'ISIC_0000276.jpg', 'ISIC_0000277.jpg', 'ISIC_0000275.jpg', 'ISIC_0000279.jpg', 'ISIC_0000287.jpg', 'ISIC_0000283.jpg', 'ISIC_0000281.jpg', 'ISIC_0000274.jpg', 'ISIC_0000303.jpg', 'ISIC_0000300.jpg', 'ISIC_0000310.jpg', 'ISIC_0000299.jpg', 'ISIC_0000301.jpg', 'ISIC_0000293.jpg', 'ISIC_0000295.jpg', 'ISIC_0000307.jpg', 'ISIC_0000311.jpg', 'ISIC_0000294.jpg', 'ISIC_0000322.jpg', 'ISIC_0000316.jpg', 'ISIC_0000319.jpg', 'ISIC_0000313.jpg', 'ISIC_0000320.jpg', 'ISIC_0000326.jpg', 'ISIC_0000325.jpg', 'ISIC_0000314.jpg', 'ISIC_0000324.jpg', 'ISIC_0000337.jpg', 'ISIC_0000329.jpg', 'ISIC_0000341.jpg', 'ISIC_0000327.jpg', 'ISIC_0000340.jpg', 'ISIC_0000336.jpg', 'ISIC_0000342.jpg', 'ISIC_0000338.jpg', 'ISIC_0000345.jpg', 'ISIC_0000349.jpg', 'ISIC_0000358.jpg', 'ISIC_0000344.jpg', 'ISIC_0000353.jpg', 'ISIC_0000354.jpg', 'ISIC_0000357.jpg', 'ISIC_0000348.jpg', 'ISIC_0000352.jpg', 'ISIC_0000351.jpg', 'ISIC_0000350.jpg', 'ISIC_0000361.jpg', 'ISIC_0000376.jpg', 'ISIC_0000366.jpg', 'ISIC_0000370.jpg', 'ISIC_0000365.jpg', 'ISIC_0000372.jpg', 'ISIC_0000367.jpg', 'ISIC_0000368.jpg', 'ISIC_0000374.jpg', 'ISIC_0000359.jpg', 'ISIC_0000363.jpg', 'ISIC_0000388.jpg', 'ISIC_0000377.jpg', 'ISIC_0000378.jpg', 'ISIC_0000385.jpg', 'ISIC_0000383.jpg', 'ISIC_0000380.jpg', 'ISIC_0000381.jpg', 'ISIC_0000386.jpg', 'ISIC_0000379.jpg', 'ISIC_0000409.jpg', 'ISIC_0000407.jpg', 'ISIC_0000397.jpg', 'ISIC_0000396.jpg', 'ISIC_0000391.jpg', 'ISIC_0000403.jpg', 'ISIC_0000410.jpg', 'ISIC_0000408.jpg', 'ISIC_0000392.jpg', 'ISIC_0000419.jpg', 'ISIC_0000416.jpg', 'ISIC_0000418.jpg', 'ISIC_0000431.jpg', 'ISIC_0000420.jpg', 'ISIC_0000425.jpg', 'ISIC_0000427.jpg', 'ISIC_0000415.jpg', 'ISIC_0000434.jpg', 'ISIC_0000433.jpg', 'ISIC_0000421.jpg', 'ISIC_0000442.jpg', 'ISIC_0000449.jpg', 'ISIC_0000448.jpg', 'ISIC_0000443.jpg', 'ISIC_0000444.jpg', 'ISIC_0000447.jpg', 'ISIC_0000439.jpg', 'ISIC_0000436.jpg', 'ISIC_0000451.jpg', 'ISIC_0000456.jpg', 'ISIC_0000463.jpg', 'ISIC_0000458.jpg', 'ISIC_0000461.jpg', 'ISIC_0000474.jpg', 'ISIC_0000476.jpg', 'ISIC_0000473.jpg', 'ISIC_0000466.jpg', 'ISIC_0000471.jpg', 'ISIC_0000470.jpg', 'ISIC_0000469.jpg', 'ISIC_0000460.jpg', 'ISIC_0000486.jpg', 'ISIC_0000487.jpg', 'ISIC_0000490.jpg', 'ISIC_0000483.jpg', 'ISIC_0000491.jpg', 'ISIC_0000481.jpg', 'ISIC_0000485.jpg', 'ISIC_0000493.jpg', 'ISIC_0000500.jpg', 'ISIC_0000495.jpg', 'ISIC_0000499.jpg', 'ISIC_0000498.jpg', 'ISIC_0000496.jpg', 'ISIC_0000503.jpg', 'ISIC_0000505.jpg', 'ISIC_0000519.jpg', 'ISIC_0000521.jpg', 'ISIC_0000515.jpg', 'ISIC_0000513.jpg', 'ISIC_0000514.jpg', 'ISIC_0000509.jpg', 'ISIC_0000510.jpg', 'ISIC_0000508.jpg', 'ISIC_0000520.jpg', 'ISIC_0000532.jpg', 'ISIC_0000536.jpg', 'ISIC_0000524.jpg', 'ISIC_0000525.jpg', 'ISIC_0000534.jpg', 'ISIC_0000531.jpg', 'ISIC_0000533.jpg', 'ISIC_0000529.jpg', 'ISIC_0000550.jpg', 'ISIC_0000540.jpg', 'ISIC_0000538.jpg', 'ISIC_0000544.jpg', 'ISIC_0000548.jpg', 'ISIC_0000541.jpg', 'ISIC_0000543.jpg', 'ISIC_0000547.jpg', 'ISIC_0000549.jpg', 'ISIC_0000542.jpg', 'ISIC_0000539.jpg', 'ISIC_0000554.jpg', 'ISIC_0001118.jpg', 'ISIC_0001106.jpg', 'ISIC_0000999.jpg', 'ISIC_0001103.jpg', 'ISIC_0000552.jpg', 'ISIC_0000882.jpg', 'ISIC_0000556.jpg', 'ISIC_0001184.jpg', 'ISIC_0001152.jpg', 'ISIC_0001119.jpg', 'ISIC_0001134.jpg', 'ISIC_0001142.jpg', 'ISIC_0001148.jpg', 'ISIC_0001140.jpg', 'ISIC_0001131.jpg', 'ISIC_0001242.jpg', 'ISIC_0001190.jpg', 'ISIC_0001185.jpg', 'ISIC_0001186.jpg', 'ISIC_0001204.jpg', 'ISIC_0001213.jpg', 'ISIC_0001188.jpg', 'ISIC_0001191.jpg', 'ISIC_0001299.jpg', 'ISIC_0001275.jpg', 'ISIC_0001296.jpg', 'ISIC_0001374.jpg', 'ISIC_0001367.jpg', 'ISIC_0001286.jpg', 'ISIC_0001292.jpg', 'ISIC_0002093.jpg', 'ISIC_0001960.jpg', 'ISIC_0002107.jpg', 'ISIC_0001769.jpg', 'ISIC_0002438.jpg', 'ISIC_0002489.jpg', 'ISIC_0002439.jpg', 'ISIC_0002780.jpg', 'ISIC_0002616.jpg', 'ISIC_0002806.jpg', 'ISIC_0002948.jpg', 'ISIC_0002647.jpg', 'ISIC_0003056.jpg', 'ISIC_0003462.jpg', 'ISIC_0003346.jpg', 'ISIC_0003539.jpg', 'ISIC_0003582.jpg', 'ISIC_0004168.jpg', 'ISIC_0004110.jpg', 'ISIC_0005247.jpg', 'ISIC_0005639.jpg', 'ISIC_0005548.jpg', 'ISIC_0005187.jpg', 'ISIC_0005666.jpg', 'ISIC_0005555.jpg', 'ISIC_0005000.jpg', 'ISIC_0006193.jpg', 'ISIC_0006612.jpg', 'ISIC_0006776.jpg', 'ISIC_0006800.jpg', 'ISIC_0006795.jpg', 'ISIC_0006711.jpg', 'ISIC_0006815.jpg', 'ISIC_0007322.jpg', 'ISIC_0006982.jpg', 'ISIC_0007475.jpg', 'ISIC_0007156.jpg']

training_attribute_streaks = ['ISIC_0013311.jpg', 'ISIC_0013837.jpg', 'ISIC_0014173.jpg', 'ISIC_0014195.jpg', 'ISIC_0014290.jpg', 'ISIC_0014667.jpg', 'ISIC_0014684.jpg', 'ISIC_0014770.jpg', 'ISIC_0014790.jpg', 'ISIC_0014823.jpg', 'ISIC_0014853.jpg', 'ISIC_0015071.jpg', 'ISIC_0015139.jpg', 'ISIC_0015133.jpg', 'ISIC_0015166.jpg', 'ISIC_0015193.jpg', 'ISIC_0015223.jpg', 'ISIC_0015980.jpg', 'ISIC_0009936.jpg', 'ISIC_0009937.jpg', 'ISIC_0009998.jpg', 'ISIC_0010016.jpg', 'ISIC_0010024.jpg', 'ISIC_0010070.jpg', 'ISIC_0010176.jpg', 'ISIC_0010175.jpg', 'ISIC_0010220.jpg', 'ISIC_0010234.jpg', 'ISIC_0010233.jpg', 'ISIC_0010256.jpg', 'ISIC_0010255.jpg', 'ISIC_0010251.jpg', 'ISIC_0010244.jpg', 'ISIC_0010329.jpg', 'ISIC_0010444.jpg', 'ISIC_0010493.jpg', 'ISIC_0010587.jpg', 'ISIC_0010599.jpg', 'ISIC_0011095.jpg', 'ISIC_0011143.jpg', 'ISIC_0011213.jpg', 'ISIC_0011210.jpg', 'ISIC_0012099.jpg', 'ISIC_0012203.jpg', 'ISIC_0012233.jpg', 'ISIC_0012413.jpg', 'ISIC_0012450.jpg', 'ISIC_0012481.jpg', 'ISIC_0012657.jpg', 'ISIC_0012544.jpg', 'ISIC_0012758.jpg', 'ISIC_0013025.jpg', 'ISIC_0013026.jpg', 'ISIC_0013147.jpg', 'ISIC_0013172.jpg', 'ISIC_0000001.jpg', 'ISIC_0000013.jpg', 'ISIC_0000034.jpg', 'ISIC_0000031.jpg', 'ISIC_0000073.jpg', 'ISIC_0000098.jpg', 'ISIC_0000103.jpg', 'ISIC_0000119.jpg', 'ISIC_0000127.jpg', 'ISIC_0000140.jpg', 'ISIC_0000138.jpg', 'ISIC_0000148.jpg', 'ISIC_0000159.jpg', 'ISIC_0000161.jpg', 'ISIC_0000155.jpg', 'ISIC_0000175.jpg', 'ISIC_0000165.jpg', 'ISIC_0000167.jpg', 'ISIC_0000173.jpg', 'ISIC_0000170.jpg', 'ISIC_0000201.jpg', 'ISIC_0000199.jpg', 'ISIC_0000200.jpg', 'ISIC_0000198.jpg', 'ISIC_0000210.jpg', 'ISIC_0000239.jpg', 'ISIC_0000240.jpg', 'ISIC_0000283.jpg', 'ISIC_0000281.jpg', 'ISIC_0000297.jpg', 'ISIC_0000342.jpg', 'ISIC_0000366.jpg', 'ISIC_0000363.jpg', 'ISIC_0000390.jpg', 'ISIC_0000431.jpg', 'ISIC_0000522.jpg', 'ISIC_0001100.jpg', 'ISIC_0001102.jpg', 'ISIC_0001103.jpg', 'ISIC_0000552.jpg', 'ISIC_0001119.jpg', 'ISIC_0001134.jpg', 'ISIC_0001142.jpg', 'ISIC_0001140.jpg', 'ISIC_0006114.jpg']

### Val

In [ ]:
val_attribute_globules = ['ISIC_0012585.jpg', 'ISIC_0012633.jpg', 'ISIC_0015351.jpg', 'ISIC_0016714.jpg', 'ISIC_0016351.jpg', 'ISIC_0015634.jpg', 'ISIC_0017460.jpg', 'ISIC_0020233.jpg', 'ISIC_0019334.jpg', 'ISIC_0020999.jpg', 'ISIC_0021152.jpg', 'ISIC_0020953.jpg', 'ISIC_0022039.jpg', 'ISIC_0023508.jpg', 'ISIC_0023904.jpg', 'ISIC_0036174.jpg', 'ISIC_0024135.jpg', 'ISIC_0036236.jpg', 'ISIC_0036321.jpg']

val_attribute_milia_like_cyst = ['ISIC_0015462.jpg', 'ISIC_0015552.jpg', 'ISIC_0017474.jpg', 'ISIC_0020953.jpg', 'ISIC_0022039.jpg', 'ISIC_0036237.jpg']

val_attribute_negative_network =  ['ISIC_0015462.jpg', 'ISIC_0015370.jpg', 'ISIC_0021202.jpg', 'ISIC_0020953.jpg', 'ISIC_0023628.jpg', 'ISIC_0036085.jpg', 'ISIC_0036147.jpg', 'ISIC_0036291.jpg', 'ISIC_0036237.jpg']

val_attribute_pigment_network =  ['ISIC_0012576.jpg', 'ISIC_0012627.jpg', 'ISIC_0012623.jpg', 'ISIC_0012585.jpg', 'ISIC_0012255.jpg', 'ISIC_0012633.jpg', 'ISIC_0015370.jpg', 'ISIC_0015351.jpg', 'ISIC_0015294.jpg', 'ISIC_0012643.jpg', 'ISIC_0015480.jpg', 'ISIC_0016714.jpg', 'ISIC_0016351.jpg', 'ISIC_0015590.jpg', 'ISIC_0015634.jpg', 'ISIC_0015492.jpg', 'ISIC_0016804.jpg', 'ISIC_0018472.jpg', 'ISIC_0017398.jpg', 'ISIC_0018111.jpg', 'ISIC_0017702.jpg', 'ISIC_0018248.jpg', 'ISIC_0017755.jpg', 'ISIC_0017399.jpg', 'ISIC_0017341.jpg', 'ISIC_0018179.jpg', 'ISIC_0017474.jpg', 'ISIC_0019723.jpg', 'ISIC_0018521.jpg', 'ISIC_0019334.jpg', 'ISIC_0019309.jpg', 'ISIC_0020418.jpg', 'ISIC_0018611.jpg', 'ISIC_0019049.jpg', 'ISIC_0019883.jpg', 'ISIC_0020999.jpg', 'ISIC_0021448.jpg', 'ISIC_0021449.jpg', 'ISIC_0021251.jpg', 'ISIC_0021152.jpg', 'ISIC_0020893.jpg', 'ISIC_0021158.jpg', 'ISIC_0021041.jpg', 'ISIC_0020861.jpg', 'ISIC_0021504.jpg', 'ISIC_0021904.jpg', 'ISIC_0022029.jpg', 'ISIC_0021762.jpg', 'ISIC_0023371.jpg', 'ISIC_0023508.jpg', 'ISIC_0022738.jpg', 'ISIC_0023678.jpg', 'ISIC_0023831.jpg', 'ISIC_0023900.jpg', 'ISIC_0022657.jpg', 'ISIC_0023904.jpg', 'ISIC_0023936.jpg', 'ISIC_0023628.jpg', 'ISIC_0036098.jpg', 'ISIC_0036085.jpg', 'ISIC_0036174.jpg', 'ISIC_0024135.jpg', 'ISIC_0036101.jpg', 'ISIC_0036206.jpg', 'ISIC_0036147.jpg', 'ISIC_0036121.jpg', 'ISIC_0036306.jpg', 'ISIC_0036321.jpg', 'ISIC_0036281.jpg', 'ISIC_0036328.jpg', 'ISIC_0036291.jpg', 'ISIC_0036240.jpg', 'ISIC_0036333.jpg']

val_attribute_streaks =  ['ISIC_0012576.jpg', 'ISIC_0017460.jpg', 'ISIC_0019794.jpg', 'ISIC_0022039.jpg', 'ISIC_0023508.jpg', 'ISIC_0036321.jpg']

### Test

In [ ]:
test_attribute_globules = ['ISIC_0012625.jpg', 'ISIC_0012591.jpg', 'ISIC_0012592.jpg', 'ISIC_0012606.jpg', 'ISIC_0012559.jpg', 'ISIC_0012650.jpg', 'ISIC_0014759.jpg', 'ISIC_0012651.jpg', 'ISIC_0014676.jpg', 'ISIC_0014953.jpg', 'ISIC_0015085.jpg', 'ISIC_0015088.jpg', 'ISIC_0015069.jpg', 'ISIC_0015121.jpg', 'ISIC_0015165.jpg', 'ISIC_0015169.jpg', 'ISIC_0015120.jpg', 'ISIC_0015473.jpg', 'ISIC_0015486.jpg', 'ISIC_0015524.jpg', 'ISIC_0015578.jpg', 'ISIC_0015587.jpg', 'ISIC_0015635.jpg', 'ISIC_0015621.jpg', 'ISIC_0017432.jpg', 'ISIC_0017415.jpg', 'ISIC_0017434.jpg', 'ISIC_0017201.jpg', 'ISIC_0017453.jpg', 'ISIC_0016736.jpg', 'ISIC_0017431.jpg', 'ISIC_0016438.jpg', 'ISIC_0017428.jpg', 'ISIC_0017389.jpg', 'ISIC_0018484.jpg', 'ISIC_0018278.jpg', 'ISIC_0017469.jpg', 'ISIC_0017775.jpg', 'ISIC_0017483.jpg', 'ISIC_0017789.jpg', 'ISIC_0018677.jpg', 'ISIC_0017486.jpg', 'ISIC_0018711.jpg', 'ISIC_0018436.jpg', 'ISIC_0019148.jpg', 'ISIC_0018819.jpg', 'ISIC_0019114.jpg', 'ISIC_0019545.jpg', 'ISIC_0019366.jpg', 'ISIC_0019335.jpg', 'ISIC_0018774.jpg', 'ISIC_0018949.jpg', 'ISIC_0019320.jpg', 'ISIC_0018917.jpg', 'ISIC_0020137.jpg', 'ISIC_0020468.jpg', 'ISIC_0019903.jpg', 'ISIC_0019736.jpg', 'ISIC_0020360.jpg', 'ISIC_0020286.jpg', 'ISIC_0020327.jpg', 'ISIC_0020443.jpg', 'ISIC_0020124.jpg', 'ISIC_0021245.jpg', 'ISIC_0021227.jpg', 'ISIC_0020972.jpg', 'ISIC_0020974.jpg', 'ISIC_0020667.jpg', 'ISIC_0021225.jpg', 'ISIC_0021008.jpg', 'ISIC_0021226.jpg', 'ISIC_0021210.jpg', 'ISIC_0021112.jpg', 'ISIC_0021187.jpg', 'ISIC_0021160.jpg', 'ISIC_0021156.jpg', 'ISIC_0021115.jpg', 'ISIC_0021103.jpg', 'ISIC_0020865.jpg', 'ISIC_0021354.jpg', 'ISIC_0021790.jpg', 'ISIC_0021751.jpg', 'ISIC_0021550.jpg', 'ISIC_0021342.jpg', 'ISIC_0021579.jpg', 'ISIC_0021791.jpg', 'ISIC_0021506.jpg', 'ISIC_0021625.jpg', 'ISIC_0021910.jpg', 'ISIC_0021893.jpg', 'ISIC_0021902.jpg', 'ISIC_0022257.jpg', 'ISIC_0022043.jpg', 'ISIC_0021800.jpg', 'ISIC_0021874.jpg', 'ISIC_0022044.jpg', 'ISIC_0022041.jpg', 'ISIC_0022776.jpg', 'ISIC_0022649.jpg', 'ISIC_0022623.jpg', 'ISIC_0022845.jpg', 'ISIC_0022329.jpg', 'ISIC_0022330.jpg', 'ISIC_0022588.jpg', 'ISIC_0022742.jpg', 'ISIC_0022735.jpg', 'ISIC_0022335.jpg', 'ISIC_0022540.jpg', 'ISIC_0022781.jpg', 'ISIC_0022736.jpg', 'ISIC_0023090.jpg', 'ISIC_0023301.jpg', 'ISIC_0023316.jpg', 'ISIC_0023205.jpg', 'ISIC_0023298.jpg', 'ISIC_0022851.jpg', 'ISIC_0022917.jpg', 'ISIC_0023101.jpg', 'ISIC_0023207.jpg', 'ISIC_0023322.jpg', 'ISIC_0023269.jpg', 'ISIC_0023321.jpg', 'ISIC_0023309.jpg', 'ISIC_0023557.jpg', 'ISIC_0023721.jpg', 'ISIC_0023506.jpg', 'ISIC_0023772.jpg', 'ISIC_0023625.jpg', 'ISIC_0023765.jpg', 'ISIC_0023472.jpg', 'ISIC_0023783.jpg', 'ISIC_0023793.jpg', 'ISIC_0023470.jpg', 'ISIC_0023940.jpg', 'ISIC_0036093.jpg', 'ISIC_0036087.jpg', 'ISIC_0036070.jpg', 'ISIC_0024157.jpg', 'ISIC_0024043.jpg', 'ISIC_0024095.jpg', 'ISIC_0036076.jpg', 'ISIC_0036083.jpg', 'ISIC_0024137.jpg', 'ISIC_0024105.jpg', 'ISIC_0024106.jpg', 'ISIC_0036125.jpg', 'ISIC_0036113.jpg', 'ISIC_0036111.jpg', 'ISIC_0036119.jpg', 'ISIC_0036114.jpg', 'ISIC_0036120.jpg', 'ISIC_0036123.jpg', 'ISIC_0036103.jpg', 'ISIC_0036124.jpg', 'ISIC_0036115.jpg', 'ISIC_0036118.jpg', 'ISIC_0036106.jpg', 'ISIC_0036126.jpg', 'ISIC_0036137.jpg', 'ISIC_0036154.jpg', 'ISIC_0036159.jpg', 'ISIC_0036145.jpg', 'ISIC_0036135.jpg', 'ISIC_0036140.jpg', 'ISIC_0036139.jpg', 'ISIC_0036146.jpg', 'ISIC_0036183.jpg', 'ISIC_0036181.jpg', 'ISIC_0036186.jpg', 'ISIC_0036187.jpg', 'ISIC_0036179.jpg', 'ISIC_0036164.jpg', 'ISIC_0036194.jpg', 'ISIC_0036191.jpg', 'ISIC_0036195.jpg', 'ISIC_0036170.jpg', 'ISIC_0036169.jpg', 'ISIC_0036211.jpg', 'ISIC_0036201.jpg', 'ISIC_0036202.jpg', 'ISIC_0036226.jpg', 'ISIC_0036224.jpg', 'ISIC_0036273.jpg', 'ISIC_0036252.jpg', 'ISIC_0036266.jpg', 'ISIC_0036242.jpg', 'ISIC_0036268.jpg', 'ISIC_0036249.jpg', 'ISIC_0036335.jpg', 'ISIC_0036329.jpg', 'ISIC_0036279.jpg', 'ISIC_0036332.jpg', 'ISIC_0036294.jpg', 'ISIC_0036318.jpg', 'ISIC_0036307.jpg', 'ISIC_0036315.jpg', 'ISIC_0036304.jpg', 'ISIC_0036302.jpg', 'ISIC_0036322.jpg', 'ISIC_0036346.jpg', 'ISIC_0036287.jpg', 'ISIC_0036297.jpg', 'ISIC_0036338.jpg', 'ISIC_0036331.jpg', 'ISIC_0036296.jpg', 'ISIC_0036309.jpg']

test_attribute_milia_like_cyst = ['ISIC_0012603.jpg', 'ISIC_0012563.jpg', 'ISIC_0012599.jpg', 'ISIC_0012606.jpg', 'ISIC_0012596.jpg', 'ISIC_0012925.jpg', 'ISIC_0012651.jpg', 'ISIC_0015001.jpg', 'ISIC_0014967.jpg', 'ISIC_0015074.jpg', 'ISIC_0015169.jpg', 'ISIC_0015275.jpg', 'ISIC_0015281.jpg', 'ISIC_0015375.jpg', 'ISIC_0015352.jpg', 'ISIC_0015398.jpg', 'ISIC_0015408.jpg', 'ISIC_0015385.jpg', 'ISIC_0015389.jpg', 'ISIC_0015435.jpg', 'ISIC_0015501.jpg', 'ISIC_0015504.jpg', 'ISIC_0015507.jpg', 'ISIC_0015521.jpg', 'ISIC_0015502.jpg', 'ISIC_0015525.jpg', 'ISIC_0015583.jpg', 'ISIC_0015613.jpg', 'ISIC_0015621.jpg', 'ISIC_0016021.jpg', 'ISIC_0016010.jpg', 'ISIC_0017413.jpg', 'ISIC_0016020.jpg', 'ISIC_0019133.jpg', 'ISIC_0020085.jpg', 'ISIC_0020601.jpg', 'ISIC_0020972.jpg', 'ISIC_0021278.jpg', 'ISIC_0021035.jpg', 'ISIC_0021196.jpg', 'ISIC_0021210.jpg', 'ISIC_0021156.jpg', 'ISIC_0021201.jpg', 'ISIC_0021551.jpg', 'ISIC_0021570.jpg', 'ISIC_0021765.jpg', 'ISIC_0021751.jpg', 'ISIC_0021701.jpg', 'ISIC_0021453.jpg', 'ISIC_0021910.jpg', 'ISIC_0021902.jpg', 'ISIC_0022217.jpg', 'ISIC_0021832.jpg', 'ISIC_0022073.jpg', 'ISIC_0022015.jpg', 'ISIC_0022261.jpg', 'ISIC_0022436.jpg', 'ISIC_0022580.jpg', 'ISIC_0022742.jpg', 'ISIC_0022741.jpg', 'ISIC_0022819.jpg', 'ISIC_0023090.jpg', 'ISIC_0022928.jpg', 'ISIC_0023013.jpg', 'ISIC_0023029.jpg', 'ISIC_0023135.jpg', 'ISIC_0023159.jpg', 'ISIC_0023832.jpg', 'ISIC_0023498.jpg', 'ISIC_0023449.jpg', 'ISIC_0023393.jpg', 'ISIC_0023468.jpg', 'ISIC_0023694.jpg', 'ISIC_0023474.jpg', 'ISIC_0024104.jpg', 'ISIC_0024139.jpg', 'ISIC_0036070.jpg', 'ISIC_0024156.jpg', 'ISIC_0036112.jpg', 'ISIC_0036132.jpg', 'ISIC_0036261.jpg', 'ISIC_0036251.jpg', 'ISIC_0036253.jpg', 'ISIC_0036301.jpg', 'ISIC_0036338.jpg']

test_attribute_negative_network = ['ISIC_0012607.jpg', 'ISIC_0014887.jpg', 'ISIC_0015088.jpg', 'ISIC_0015504.jpg', 'ISIC_0015570.jpg', 'ISIC_0017434.jpg', 'ISIC_0017392.jpg', 'ISIC_0018040.jpg', 'ISIC_0017987.jpg', 'ISIC_0017483.jpg', 'ISIC_0018627.jpg', 'ISIC_0018200.jpg', 'ISIC_0017675.jpg', 'ISIC_0018039.jpg', 'ISIC_0018345.jpg', 'ISIC_0018058.jpg', 'ISIC_0018436.jpg', 'ISIC_0019366.jpg', 'ISIC_0018989.jpg', 'ISIC_0018941.jpg', 'ISIC_0019432.jpg', 'ISIC_0019030.jpg', 'ISIC_0020085.jpg', 'ISIC_0020420.jpg', 'ISIC_0020548.jpg', 'ISIC_0020549.jpg', 'ISIC_0020951.jpg', 'ISIC_0021187.jpg', 'ISIC_0020632.jpg', 'ISIC_0021751.jpg', 'ISIC_0021406.jpg', 'ISIC_0021701.jpg', 'ISIC_0021738.jpg', 'ISIC_0022009.jpg', 'ISIC_0022776.jpg', 'ISIC_0022546.jpg', 'ISIC_0022308.jpg', 'ISIC_0022591.jpg', 'ISIC_0023203.jpg', 'ISIC_0023226.jpg', 'ISIC_0023201.jpg', 'ISIC_0023202.jpg', 'ISIC_0023221.jpg', 'ISIC_0023553.jpg', 'ISIC_0023761.jpg', 'ISIC_0023481.jpg', 'ISIC_0023784.jpg', 'ISIC_0023783.jpg', 'ISIC_0023779.jpg', 'ISIC_0024104.jpg', 'ISIC_0036092.jpg', 'ISIC_0023911.jpg', 'ISIC_0036091.jpg', 'ISIC_0023912.jpg', 'ISIC_0036079.jpg', 'ISIC_0024137.jpg', 'ISIC_0036112.jpg', 'ISIC_0036130.jpg', 'ISIC_0036152.jpg', 'ISIC_0036142.jpg', 'ISIC_0036179.jpg', 'ISIC_0036190.jpg', 'ISIC_0036194.jpg', 'ISIC_0036175.jpg', 'ISIC_0036224.jpg', 'ISIC_0036273.jpg', 'ISIC_0036266.jpg', 'ISIC_0036234.jpg', 'ISIC_0036264.jpg', 'ISIC_0036269.jpg', 'ISIC_0036276.jpg', 'ISIC_0036326.jpg', 'ISIC_0036280.jpg', 'ISIC_0036287.jpg', 'ISIC_0036297.jpg', 'ISIC_0036309.jpg']

test_attribute_pigment_network = ['ISIC_0012634.jpg', 'ISIC_0012292.jpg', 'ISIC_0012591.jpg', 'ISIC_0012337.jpg', 'ISIC_0012603.jpg', 'ISIC_0012302.jpg', 'ISIC_0012560.jpg', 'ISIC_0012563.jpg', 'ISIC_0012599.jpg', 'ISIC_0012418.jpg', 'ISIC_0012596.jpg', 'ISIC_0012611.jpg', 'ISIC_0012647.jpg', 'ISIC_0012236.jpg', 'ISIC_0013143.jpg', 'ISIC_0014759.jpg', 'ISIC_0012925.jpg', 'ISIC_0012651.jpg', 'ISIC_0014888.jpg', 'ISIC_0014905.jpg', 'ISIC_0014887.jpg', 'ISIC_0012698.jpg', 'ISIC_0012935.jpg', 'ISIC_0014758.jpg', 'ISIC_0015001.jpg', 'ISIC_0014906.jpg', 'ISIC_0014953.jpg', 'ISIC_0014976.jpg', 'ISIC_0014997.jpg', 'ISIC_0014967.jpg', 'ISIC_0015094.jpg', 'ISIC_0015085.jpg', 'ISIC_0015074.jpg', 'ISIC_0015088.jpg', 'ISIC_0015104.jpg', 'ISIC_0015117.jpg', 'ISIC_0015121.jpg', 'ISIC_0015137.jpg', 'ISIC_0015154.jpg', 'ISIC_0015165.jpg', 'ISIC_0015169.jpg', 'ISIC_0015120.jpg', 'ISIC_0015227.jpg', 'ISIC_0015275.jpg', 'ISIC_0015265.jpg', 'ISIC_0015290.jpg', 'ISIC_0015257.jpg', 'ISIC_0015352.jpg', 'ISIC_0015350.jpg', 'ISIC_0015365.jpg', 'ISIC_0015366.jpg', 'ISIC_0015408.jpg', 'ISIC_0015381.jpg', 'ISIC_0015463.jpg', 'ISIC_0015451.jpg', 'ISIC_0015450.jpg', 'ISIC_0015448.jpg', 'ISIC_0015501.jpg', 'ISIC_0015469.jpg', 'ISIC_0015470.jpg', 'ISIC_0015478.jpg', 'ISIC_0015520.jpg', 'ISIC_0015504.jpg', 'ISIC_0015507.jpg', 'ISIC_0015524.jpg', 'ISIC_0015521.jpg', 'ISIC_0015525.jpg', 'ISIC_0015583.jpg', 'ISIC_0015569.jpg', 'ISIC_0015570.jpg', 'ISIC_0015578.jpg', 'ISIC_0015970.jpg', 'ISIC_0015635.jpg', 'ISIC_0015602.jpg', 'ISIC_0015637.jpg', 'ISIC_0015629.jpg', 'ISIC_0016363.jpg', 'ISIC_0017436.jpg', 'ISIC_0017422.jpg', 'ISIC_0016197.jpg', 'ISIC_0016364.jpg', 'ISIC_0017415.jpg', 'ISIC_0016274.jpg', 'ISIC_0017350.jpg', 'ISIC_0016524.jpg', 'ISIC_0016963.jpg', 'ISIC_0017443.jpg', 'ISIC_0017412.jpg', 'ISIC_0016186.jpg', 'ISIC_0017433.jpg', 'ISIC_0016199.jpg', 'ISIC_0017449.jpg', 'ISIC_0016736.jpg', 'ISIC_0016575.jpg', 'ISIC_0017451.jpg', 'ISIC_0017392.jpg', 'ISIC_0016021.jpg', 'ISIC_0017409.jpg', 'ISIC_0017348.jpg', 'ISIC_0016211.jpg', 'ISIC_0017264.jpg', 'ISIC_0016010.jpg', 'ISIC_0016911.jpg', 'ISIC_0017292.jpg', 'ISIC_0017026.jpg', 'ISIC_0016726.jpg', 'ISIC_0017413.jpg', 'ISIC_0017444.jpg', 'ISIC_0017428.jpg', 'ISIC_0017427.jpg', 'ISIC_0017389.jpg', 'ISIC_0016318.jpg', 'ISIC_0017259.jpg', 'ISIC_0017465.jpg', 'ISIC_0018040.jpg', 'ISIC_0018424.jpg', 'ISIC_0018377.jpg', 'ISIC_0017485.jpg', 'ISIC_0018405.jpg', 'ISIC_0018278.jpg', 'ISIC_0017469.jpg', 'ISIC_0018109.jpg', 'ISIC_0018662.jpg', 'ISIC_0018580.jpg', 'ISIC_0018602.jpg', 'ISIC_0017775.jpg', 'ISIC_0017704.jpg', 'ISIC_0017693.jpg', 'ISIC_0017990.jpg', 'ISIC_0017987.jpg', 'ISIC_0018527.jpg', 'ISIC_0017970.jpg', 'ISIC_0018603.jpg', 'ISIC_0018325.jpg', 'ISIC_0018090.jpg', 'ISIC_0018324.jpg', 'ISIC_0017789.jpg', 'ISIC_0018339.jpg', 'ISIC_0018593.jpg', 'ISIC_0018677.jpg', 'ISIC_0018336.jpg', 'ISIC_0017883.jpg', 'ISIC_0017891.jpg', 'ISIC_0018507.jpg', 'ISIC_0017993.jpg', 'ISIC_0017793.jpg', 'ISIC_0017856.jpg', 'ISIC_0018200.jpg', 'ISIC_0017675.jpg', 'ISIC_0018392.jpg', 'ISIC_0018413.jpg', 'ISIC_0018535.jpg', 'ISIC_0018088.jpg', 'ISIC_0018415.jpg', 'ISIC_0018039.jpg', 'ISIC_0018209.jpg', 'ISIC_0018185.jpg', 'ISIC_0018297.jpg', 'ISIC_0017750.jpg', 'ISIC_0018711.jpg', 'ISIC_0018286.jpg', 'ISIC_0017939.jpg', 'ISIC_0017689.jpg', 'ISIC_0018573.jpg', 'ISIC_0018126.jpg', 'ISIC_0018184.jpg', 'ISIC_0017479.jpg', 'ISIC_0018633.jpg', 'ISIC_0018048.jpg', 'ISIC_0018108.jpg', 'ISIC_0018371.jpg', 'ISIC_0017951.jpg', 'ISIC_0017973.jpg', 'ISIC_0018960.jpg', 'ISIC_0019016.jpg', 'ISIC_0018819.jpg', 'ISIC_0019092.jpg', 'ISIC_0019545.jpg', 'ISIC_0019529.jpg', 'ISIC_0019366.jpg', 'ISIC_0019090.jpg', 'ISIC_0019143.jpg', 'ISIC_0019178.jpg', 'ISIC_0019228.jpg', 'ISIC_0018744.jpg', 'ISIC_0019135.jpg', 'ISIC_0019088.jpg', 'ISIC_0019627.jpg', 'ISIC_0019424.jpg', 'ISIC_0019540.jpg', 'ISIC_0019133.jpg', 'ISIC_0019528.jpg', 'ISIC_0018964.jpg', 'ISIC_0018747.jpg', 'ISIC_0019127.jpg', 'ISIC_0019380.jpg', 'ISIC_0019260.jpg', 'ISIC_0019385.jpg', 'ISIC_0018889.jpg', 'ISIC_0018799.jpg', 'ISIC_0018774.jpg', 'ISIC_0019512.jpg', 'ISIC_0019062.jpg', 'ISIC_0019200.jpg', 'ISIC_0019609.jpg', 'ISIC_0019481.jpg', 'ISIC_0019544.jpg', 'ISIC_0019315.jpg', 'ISIC_0019364.jpg', 'ISIC_0019532.jpg', 'ISIC_0019354.jpg', 'ISIC_0019307.jpg', 'ISIC_0019204.jpg', 'ISIC_0019326.jpg', 'ISIC_0019170.jpg', 'ISIC_0019463.jpg', 'ISIC_0019173.jpg', 'ISIC_0019423.jpg', 'ISIC_0019320.jpg', 'ISIC_0018727.jpg', 'ISIC_0019281.jpg', 'ISIC_0018734.jpg', 'ISIC_0018738.jpg', 'ISIC_0019324.jpg', 'ISIC_0019313.jpg', 'ISIC_0018766.jpg', 'ISIC_0019030.jpg', 'ISIC_0019303.jpg', 'ISIC_0019453.jpg', 'ISIC_0019312.jpg', 'ISIC_0018929.jpg', 'ISIC_0018917.jpg', 'ISIC_0020185.jpg', 'ISIC_0020137.jpg', 'ISIC_0020166.jpg', 'ISIC_0020235.jpg', 'ISIC_0020468.jpg', 'ISIC_0020491.jpg', 'ISIC_0020409.jpg', 'ISIC_0019903.jpg', 'ISIC_0019981.jpg', 'ISIC_0020359.jpg', 'ISIC_0020454.jpg', 'ISIC_0019664.jpg', 'ISIC_0019911.jpg', 'ISIC_0020300.jpg', 'ISIC_0020373.jpg', 'ISIC_0020613.jpg', 'ISIC_0020550.jpg', 'ISIC_0020135.jpg', 'ISIC_0020366.jpg', 'ISIC_0019696.jpg', 'ISIC_0020154.jpg', 'ISIC_0020307.jpg', 'ISIC_0019870.jpg', 'ISIC_0019989.jpg', 'ISIC_0019736.jpg', 'ISIC_0019669.jpg', 'ISIC_0019964.jpg', 'ISIC_0020327.jpg', 'ISIC_0019678.jpg', 'ISIC_0020507.jpg', 'ISIC_0020443.jpg', 'ISIC_0020420.jpg', 'ISIC_0020372.jpg', 'ISIC_0019935.jpg', 'ISIC_0020608.jpg', 'ISIC_0019703.jpg', 'ISIC_0020548.jpg', 'ISIC_0020081.jpg', 'ISIC_0020334.jpg', 'ISIC_0020318.jpg', 'ISIC_0020540.jpg', 'ISIC_0019910.jpg', 'ISIC_0020353.jpg', 'ISIC_0020598.jpg', 'ISIC_0020009.jpg', 'ISIC_0020531.jpg', 'ISIC_0020479.jpg', 'ISIC_0019902.jpg', 'ISIC_0020500.jpg', 'ISIC_0019816.jpg', 'ISIC_0019628.jpg', 'ISIC_0019695.jpg', 'ISIC_0020601.jpg', 'ISIC_0019992.jpg', 'ISIC_0019733.jpg', 'ISIC_0020508.jpg', 'ISIC_0020004.jpg', 'ISIC_0019638.jpg', 'ISIC_0020150.jpg', 'ISIC_0019970.jpg', 'ISIC_0020226.jpg', 'ISIC_0020549.jpg', 'ISIC_0020005.jpg', 'ISIC_0020951.jpg', 'ISIC_0020626.jpg', 'ISIC_0021270.jpg', 'ISIC_0020860.jpg', 'ISIC_0021227.jpg', 'ISIC_0021070.jpg', 'ISIC_0020896.jpg', 'ISIC_0021054.jpg', 'ISIC_0020620.jpg', 'ISIC_0020619.jpg', 'ISIC_0020743.jpg', 'ISIC_0021114.jpg', 'ISIC_0020895.jpg', 'ISIC_0020962.jpg', 'ISIC_0020714.jpg', 'ISIC_0020667.jpg', 'ISIC_0020663.jpg', 'ISIC_0020798.jpg', 'ISIC_0021051.jpg', 'ISIC_0021157.jpg', 'ISIC_0021050.jpg', 'ISIC_0021260.jpg', 'ISIC_0020737.jpg', 'ISIC_0021226.jpg', 'ISIC_0020764.jpg', 'ISIC_0021112.jpg', 'ISIC_0020987.jpg', 'ISIC_0021263.jpg', 'ISIC_0020935.jpg', 'ISIC_0020803.jpg', 'ISIC_0020840.jpg', 'ISIC_0021160.jpg', 'ISIC_0021092.jpg', 'ISIC_0021156.jpg', 'ISIC_0020683.jpg', 'ISIC_0020921.jpg', 'ISIC_0021115.jpg', 'ISIC_0020879.jpg', 'ISIC_0021103.jpg', 'ISIC_0020640.jpg', 'ISIC_0020632.jpg', 'ISIC_0020761.jpg', 'ISIC_0021551.jpg', 'ISIC_0021622.jpg', 'ISIC_0021451.jpg', 'ISIC_0021777.jpg', 'ISIC_0021618.jpg', 'ISIC_0021596.jpg', 'ISIC_0021593.jpg', 'ISIC_0021765.jpg', 'ISIC_0021319.jpg', 'ISIC_0021354.jpg', 'ISIC_0021790.jpg', 'ISIC_0021751.jpg', 'ISIC_0021333.jpg', 'ISIC_0021406.jpg', 'ISIC_0021659.jpg', 'ISIC_0021550.jpg', 'ISIC_0021558.jpg', 'ISIC_0021342.jpg', 'ISIC_0021573.jpg', 'ISIC_0021494.jpg', 'ISIC_0021791.jpg', 'ISIC_0021753.jpg', 'ISIC_0021688.jpg', 'ISIC_0021452.jpg', 'ISIC_0021480.jpg', 'ISIC_0021559.jpg', 'ISIC_0021616.jpg', 'ISIC_0021506.jpg', 'ISIC_0021374.jpg', 'ISIC_0021604.jpg', 'ISIC_0021734.jpg', 'ISIC_0021648.jpg', 'ISIC_0021591.jpg', 'ISIC_0021453.jpg', 'ISIC_0021407.jpg', 'ISIC_0021786.jpg', 'ISIC_0021380.jpg', 'ISIC_0021447.jpg', 'ISIC_0021379.jpg', 'ISIC_0021313.jpg', 'ISIC_0021554.jpg', 'ISIC_0021845.jpg', 'ISIC_0022247.jpg', 'ISIC_0021902.jpg', 'ISIC_0022208.jpg', 'ISIC_0022210.jpg', 'ISIC_0022213.jpg', 'ISIC_0021932.jpg', 'ISIC_0021955.jpg', 'ISIC_0022217.jpg', 'ISIC_0021837.jpg', 'ISIC_0022028.jpg', 'ISIC_0021874.jpg', 'ISIC_0022006.jpg', 'ISIC_0022025.jpg', 'ISIC_0021940.jpg', 'ISIC_0022009.jpg', 'ISIC_0021956.jpg', 'ISIC_0022245.jpg', 'ISIC_0021930.jpg', 'ISIC_0022215.jpg', 'ISIC_0021999.jpg', 'ISIC_0021987.jpg', 'ISIC_0022073.jpg', 'ISIC_0021921.jpg', 'ISIC_0021959.jpg', 'ISIC_0021873.jpg', 'ISIC_0021988.jpg', 'ISIC_0022191.jpg', 'ISIC_0022551.jpg', 'ISIC_0022570.jpg', 'ISIC_0022844.jpg', 'ISIC_0022845.jpg', 'ISIC_0022546.jpg', 'ISIC_0022329.jpg', 'ISIC_0022330.jpg', 'ISIC_0022572.jpg', 'ISIC_0022317.jpg', 'ISIC_0022381.jpg', 'ISIC_0022733.jpg', 'ISIC_0022367.jpg', 'ISIC_0022308.jpg', 'ISIC_0022552.jpg', 'ISIC_0022828.jpg', 'ISIC_0022550.jpg', 'ISIC_0022732.jpg', 'ISIC_0022574.jpg', 'ISIC_0022820.jpg', 'ISIC_0022663.jpg', 'ISIC_0022819.jpg', 'ISIC_0022662.jpg', 'ISIC_0022730.jpg', 'ISIC_0022335.jpg', 'ISIC_0022571.jpg', 'ISIC_0022540.jpg', 'ISIC_0022444.jpg', 'ISIC_0023090.jpg', 'ISIC_0023301.jpg', 'ISIC_0023235.jpg', 'ISIC_0022849.jpg', 'ISIC_0023199.jpg', 'ISIC_0023274.jpg', 'ISIC_0023232.jpg', 'ISIC_0023316.jpg', 'ISIC_0023156.jpg', 'ISIC_0023306.jpg', 'ISIC_0023118.jpg', 'ISIC_0023211.jpg', 'ISIC_0023013.jpg', 'ISIC_0023202.jpg', 'ISIC_0023343.jpg', 'ISIC_0023298.jpg', 'ISIC_0023164.jpg', 'ISIC_0023261.jpg', 'ISIC_0022851.jpg', 'ISIC_0023264.jpg', 'ISIC_0023101.jpg', 'ISIC_0023322.jpg', 'ISIC_0023269.jpg', 'ISIC_0023056.jpg', 'ISIC_0023283.jpg', 'ISIC_0023300.jpg', 'ISIC_0023832.jpg', 'ISIC_0023721.jpg', 'ISIC_0023506.jpg', 'ISIC_0023804.jpg', 'ISIC_0023772.jpg', 'ISIC_0023747.jpg', 'ISIC_0023379.jpg', 'ISIC_0023821.jpg', 'ISIC_0023449.jpg', 'ISIC_0023658.jpg', 'ISIC_0023696.jpg', 'ISIC_0023575.jpg', 'ISIC_0023761.jpg', 'ISIC_0023346.jpg', 'ISIC_0023646.jpg', 'ISIC_0023625.jpg', 'ISIC_0023715.jpg', 'ISIC_0023366.jpg', 'ISIC_0023481.jpg', 'ISIC_0023765.jpg', 'ISIC_0023358.jpg', 'ISIC_0023356.jpg', 'ISIC_0023468.jpg', 'ISIC_0023812.jpg', 'ISIC_0023752.jpg', 'ISIC_0023784.jpg', 'ISIC_0023813.jpg', 'ISIC_0023396.jpg', 'ISIC_0023472.jpg', 'ISIC_0023364.jpg', 'ISIC_0023783.jpg', 'ISIC_0023620.jpg', 'ISIC_0023762.jpg', 'ISIC_0023793.jpg', 'ISIC_0023676.jpg', 'ISIC_0023474.jpg', 'ISIC_0023779.jpg', 'ISIC_0023470.jpg', 'ISIC_0024104.jpg', 'ISIC_0036092.jpg', 'ISIC_0023911.jpg', 'ISIC_0024100.jpg', 'ISIC_0023881.jpg', 'ISIC_0023920.jpg', 'ISIC_0023948.jpg', 'ISIC_0023901.jpg', 'ISIC_0023933.jpg', 'ISIC_0023912.jpg', 'ISIC_0024032.jpg', 'ISIC_0024095.jpg', 'ISIC_0024130.jpg', 'ISIC_0036076.jpg', 'ISIC_0036079.jpg', 'ISIC_0023885.jpg', 'ISIC_0024106.jpg', 'ISIC_0024005.jpg', 'ISIC_0036097.jpg', 'ISIC_0036117.jpg', 'ISIC_0036111.jpg', 'ISIC_0036128.jpg', 'ISIC_0036119.jpg', 'ISIC_0036114.jpg', 'ISIC_0036127.jpg', 'ISIC_0036100.jpg', 'ISIC_0036116.jpg', 'ISIC_0036120.jpg', 'ISIC_0036109.jpg', 'ISIC_0036103.jpg', 'ISIC_0036104.jpg', 'ISIC_0036096.jpg', 'ISIC_0036115.jpg', 'ISIC_0036122.jpg', 'ISIC_0036118.jpg', 'ISIC_0036108.jpg', 'ISIC_0036110.jpg', 'ISIC_0036133.jpg', 'ISIC_0036134.jpg', 'ISIC_0036137.jpg', 'ISIC_0036138.jpg', 'ISIC_0036154.jpg', 'ISIC_0036157.jpg', 'ISIC_0036130.jpg', 'ISIC_0036159.jpg', 'ISIC_0036158.jpg', 'ISIC_0036145.jpg', 'ISIC_0036160.jpg', 'ISIC_0036136.jpg', 'ISIC_0036135.jpg', 'ISIC_0036140.jpg', 'ISIC_0036152.jpg', 'ISIC_0036142.jpg', 'ISIC_0036151.jpg', 'ISIC_0036129.jpg', 'ISIC_0036132.jpg', 'ISIC_0036131.jpg', 'ISIC_0036149.jpg', 'ISIC_0036146.jpg', 'ISIC_0036150.jpg', 'ISIC_0036141.jpg', 'ISIC_0036192.jpg', 'ISIC_0036168.jpg', 'ISIC_0036181.jpg', 'ISIC_0036167.jpg', 'ISIC_0036166.jpg', 'ISIC_0036193.jpg', 'ISIC_0036186.jpg', 'ISIC_0036187.jpg', 'ISIC_0036179.jpg', 'ISIC_0036164.jpg', 'ISIC_0036184.jpg', 'ISIC_0036178.jpg', 'ISIC_0036188.jpg', 'ISIC_0036175.jpg', 'ISIC_0036177.jpg', 'ISIC_0036185.jpg', 'ISIC_0036211.jpg', 'ISIC_0036196.jpg', 'ISIC_0036205.jpg', 'ISIC_0036208.jpg', 'ISIC_0036221.jpg', 'ISIC_0036222.jpg', 'ISIC_0036225.jpg', 'ISIC_0036217.jpg', 'ISIC_0036220.jpg', 'ISIC_0036212.jpg', 'ISIC_0036218.jpg', 'ISIC_0036213.jpg', 'ISIC_0036197.jpg', 'ISIC_0036204.jpg', 'ISIC_0036198.jpg', 'ISIC_0036235.jpg', 'ISIC_0036260.jpg', 'ISIC_0036230.jpg', 'ISIC_0036259.jpg', 'ISIC_0036241.jpg', 'ISIC_0036265.jpg', 'ISIC_0036263.jpg', 'ISIC_0036254.jpg', 'ISIC_0036251.jpg', 'ISIC_0036273.jpg', 'ISIC_0036238.jpg', 'ISIC_0036232.jpg', 'ISIC_0036266.jpg', 'ISIC_0036242.jpg', 'ISIC_0036268.jpg', 'ISIC_0036262.jpg', 'ISIC_0036239.jpg', 'ISIC_0036270.jpg', 'ISIC_0036274.jpg', 'ISIC_0036233.jpg', 'ISIC_0036231.jpg', 'ISIC_0036264.jpg', 'ISIC_0036243.jpg', 'ISIC_0036347.jpg', 'ISIC_0036341.jpg', 'ISIC_0036336.jpg', 'ISIC_0036292.jpg', 'ISIC_0036312.jpg', 'ISIC_0036337.jpg', 'ISIC_0036310.jpg', 'ISIC_0036279.jpg', 'ISIC_0036288.jpg', 'ISIC_0036295.jpg', 'ISIC_0036308.jpg', 'ISIC_0036318.jpg', 'ISIC_0036285.jpg', 'ISIC_0036277.jpg', 'ISIC_0036326.jpg', 'ISIC_0036290.jpg', 'ISIC_0036298.jpg', 'ISIC_0036304.jpg', 'ISIC_0036303.jpg', 'ISIC_0036300.jpg', 'ISIC_0036278.jpg', 'ISIC_0036340.jpg', 'ISIC_0036302.jpg', 'ISIC_0036305.jpg', 'ISIC_0036280.jpg', 'ISIC_0036343.jpg', 'ISIC_0036346.jpg', 'ISIC_0036319.jpg', 'ISIC_0036297.jpg', 'ISIC_0036327.jpg', 'ISIC_0036331.jpg', 'ISIC_0036345.jpg', 'ISIC_0036342.jpg', 'ISIC_0036286.jpg', 'ISIC_0036293.jpg', 'ISIC_0036323.jpg']

test_attribute_streaks = ['ISIC_0012591.jpg', 'ISIC_0014887.jpg', 'ISIC_0012698.jpg', 'ISIC_0012935.jpg', 'ISIC_0014997.jpg', 'ISIC_0015479.jpg', 'ISIC_0015551.jpg', 'ISIC_0015520.jpg', 'ISIC_0015578.jpg', 'ISIC_0017432.jpg', 'ISIC_0017436.jpg', 'ISIC_0017422.jpg', 'ISIC_0017406.jpg', 'ISIC_0017448.jpg', 'ISIC_0017412.jpg', 'ISIC_0017453.jpg', 'ISIC_0017449.jpg', 'ISIC_0017431.jpg', 'ISIC_0017409.jpg', 'ISIC_0016663.jpg', 'ISIC_0017385.jpg', 'ISIC_0016911.jpg', 'ISIC_0017413.jpg', 'ISIC_0017465.jpg', 'ISIC_0017485.jpg', 'ISIC_0017486.jpg', 'ISIC_0017479.jpg', 'ISIC_0018633.jpg', 'ISIC_0020468.jpg', 'ISIC_0020663.jpg', 'ISIC_0021170.jpg', 'ISIC_0021347.jpg', 'ISIC_0021601.jpg', 'ISIC_0021494.jpg', 'ISIC_0021647.jpg', 'ISIC_0021648.jpg', 'ISIC_0021591.jpg', 'ISIC_0022257.jpg', 'ISIC_0022043.jpg', 'ISIC_0022044.jpg', 'ISIC_0021940.jpg', 'ISIC_0022041.jpg', 'ISIC_0022367.jpg', 'ISIC_0023301.jpg', 'ISIC_0023316.jpg', 'ISIC_0023264.jpg', 'ISIC_0023322.jpg', 'ISIC_0023269.jpg', 'ISIC_0023128.jpg', 'ISIC_0023309.jpg', 'ISIC_0023812.jpg', 'ISIC_0023783.jpg', 'ISIC_0023916.jpg', 'ISIC_0036093.jpg', 'ISIC_0036066.jpg', 'ISIC_0036128.jpg', 'ISIC_0036119.jpg', 'ISIC_0036114.jpg', 'ISIC_0036116.jpg', 'ISIC_0036120.jpg', 'ISIC_0036104.jpg', 'ISIC_0036150.jpg', 'ISIC_0036166.jpg', 'ISIC_0036171.jpg', 'ISIC_0036169.jpg', 'ISIC_0036223.jpg', 'ISIC_0036221.jpg', 'ISIC_0036228.jpg', 'ISIC_0036215.jpg', 'ISIC_0036263.jpg', 'ISIC_0036266.jpg', 'ISIC_0036329.jpg', 'ISIC_0036295.jpg', 'ISIC_0036294.jpg', 'ISIC_0036307.jpg', 'ISIC_0036277.jpg', 'ISIC_0036344.jpg']


In [ ]:
print(len(test_attribute_globules))
print(len(test_attribute_milia_like_cyst))
print(len(test_attribute_negative_network))
print(len(test_attribute_pigment_network))
print(len(test_attribute_streaks))

206
85
76
648
77


training_attribute_globules: 603
training_attribute_milia_like_cyst: 682
training_attribute_negative_network: 190
training_attribute_pigment_network: 1523
training_attribute_streaks: 100

TOTAL: 3098 (2594 imagens) -> 504 imagens contendo mais de uma classe



val_attribute_pigment_network: 73
val_attribute_globules: 19
val_attribute_negative_network: 9
val_attribute_milia_like_cyst: 6
val_attribute_streaks: 6


TOTAL: 113 (100 imagens)-> 13 imagens contendo mais de uma classe

## Results:

### Training mask: (2594 imagens)

* attribute_globules: 23.09% (603 imagens)
* attribute_milia_like_cyst: 26,38% (682 imagens)
* attribute_negative_network: 7.28% (190 imagens)
* attribute_pigment_network: 58,77% (1523 imagens)
* attribute_streaks: 3,845% (100 imagens)

Media: 619.6 (600 imagens)

### Validation mask: (100 imagens)

* attribute_globules: 19%(19 imagens)
* attribute_milia_like_cyst: 6% (6 imagens)
* attribute_negative_network: 9% (9 imagens)
* attribute_pigment_network: 73% (73 imagens)
* attribute_streaks: 6% (6 imagens)

Media: 22.6 (20 imagens)


### Validation mask: (1000 imagens)

* attribute_globules: 20,6%(206 imagens)
* attribute_milia_like_cyst: 8,5% (85 imagens)
* attribute_negative_network: 7,6% (76 imagens)
* attribute_pigment_network: 64,8%% (648 imagens)
* attribute_streaks: 7,7% (77 imagens)

Media: 218.4 (200 imagens)